<!DOCTYPE html><html lang="en">
<head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<title>Non-subjective power analysis to detect G×E interactions in Genome-Wide Association Studies in presence of confounding factor</title>
<!--Generated on Thu Dec  5 02:14:01 2024 by LaTeXML (version 0.8.8) http://dlmf.nist.gov/LaTeXML/.-->

<link rel="stylesheet" href="LaTeXML.css" type="text/css">
<link rel="stylesheet" href="ltx-article.css" type="text/css">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
</head>
<body>
<div class="ltx_page_main">
<div class="ltx_page_content">
<article class="ltx_document ltx_authors_1line" lang="en">
<h1 class="ltx_title ltx_title_document">Non-subjective power analysis to detect G<math id="m1" class="ltx_Math" alttext="\times" display="inline"><mo>×</mo></math>E interactions in Genome-Wide Association Studies in presence of confounding factor</h1>
<div class="ltx_authors">
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">F. Alarcon<sup class="ltx_sup">1</sup>, V. Perduca<sup class="ltx_sup">1</sup>, G. Nuel<sup class="ltx_sup">2</sup>
</span></span>
</div>

<div class="ltx_abstract">
<h6 class="ltx_title ltx_title_abstract">Abstract</h6>
    
<p class="ltx_p"><span class="ltx_text ltx_font_bold">Background.<span class="ltx_text ltx_font_medium"> It is generally acknowledged that most complex diseases are affected in part by interactions between genes and genes and/or between genes and environmental factors. Taking into account environmental exposures and their interactions with genetic factors in genome-wide association studies (GWAS) can help to identify high-risk subgroups in the population and provide a better understanding of the disease. For this reason, many methods have been developed to detect gene-environment (G<math id="m5" class="ltx_Math" alttext="\times" display="inline"><mo>×</mo></math>E) interactions. Despite this, few loci that interact with environmental exposures have been identified so far. Indeed, the modest effect of G<math id="m6" class="ltx_Math" alttext="\times" display="inline"><mo>×</mo></math>E interactions as well as confounding factors entail low statistical power to detect such interactions.</span></span></p>
    
<p class="ltx_p"><span class="ltx_text ltx_font_bold">Results.<span class="ltx_text ltx_font_medium">
In this work, we provide a simulated dataset in order to study methods for detecting G<math id="m7" class="ltx_Math" alttext="\times" display="inline"><mo>×</mo></math>E interactions in GWAS in presence of confounding factor and population structure. Our work applies a recently introduced non-subjective method for H1 simulations called <math id="m8" class="ltx_Math" alttext="{\tt waffect}" display="inline"><mi>𝚠𝚊𝚏𝚏𝚎𝚌𝚝</mi></math> and exploits the publicly available HapMap project to build a datasets with real genotypes and population structures. We use this dataset to study the impact of confounding factors and compare the relative performance of popular methods such as PLINK, random forests and linear mixed models to detect G<math id="m9" class="ltx_Math" alttext="\times" display="inline"><mo>×</mo></math>E interactions.</span></span></p>
    
<p class="ltx_p"><span class="ltx_text ltx_font_bold">Conclusion.<span class="ltx_text ltx_font_medium">
Presence of confounding factor is an obstacle to detect G<math id="m10" class="ltx_Math" alttext="\times" display="inline"><mo>×</mo></math>E interactions in GWAS and the approaches considered in our power study all have insufficient power to detect the strong simulated interaction.
Our simulated dataset could help to develop new methods which account for confounding factors through latent exposures in order to improve power.</span></span></p>
  
</div>
<div id="p1" class="ltx_para">
<p class="ltx_p"><sup class="ltx_sup">1</sup> MAP5-UMR CNRS 8145, Sorbonne Paris Cité University, Paris, France.
<br class="ltx_break"><sup class="ltx_sup">2</sup> LPMA, UPMC, Sorbonne University, Paris, France.
<br class="ltx_break"></p>
</div>
<div class="ltx_pagination ltx_role_newpage"></div>
<section id="S1" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">1 </span>Introduction</h2>

<div id="S1.p1" class="ltx_para">
<p class="ltx_p">Genome-wide association studies (GWAS) are a standard method to identify common genetic factors that influence health and disease conditions. These methods have improved our understanding of the genetic basis of many complex traits and are among the most used tools for analyzing complex diseases.
However, it is known that most complex diseases (e.g. diabetes, asthma and cancer) are due to combined effect of genes, environmental factors, as well as their interactions <cite class="ltx_cite ltx_citemacro_citep">(Murcray et al., <a href="#bib.bib17" title="" class="ltx_ref">2009</a>)</cite>.</p>
</div>
<div id="S1.p2" class="ltx_para">
<p class="ltx_p">Over the last years, considerable efforts have been put to detect gene-environment interactions (G<math id="S1.p2.m1" class="ltx_Math" alttext="\times" display="inline"><mo>×</mo></math>E) in GWAS and few loci that interact with environmental exposures have been identified <cite class="ltx_cite ltx_citemacro_citep">(Rothman et al., <a href="#bib.bib22" title="" class="ltx_ref">2010</a>; Hamza et al., <a href="#bib.bib11" title="" class="ltx_ref">2011</a>; Garcia-Closas et al., <a href="#bib.bib8" title="" class="ltx_ref">2010</a>)</cite>.</p>
</div>
<div id="S1.p3" class="ltx_para">
<p class="ltx_p">However, this problem is well known to be challenging due, in part, to the modest effect of such interactions in terms of relative risk.
Another reason why detecting G<math id="S1.p3.m1" class="ltx_Math" alttext="\times" display="inline"><mo>×</mo></math>E interactions is difficult is population structure which can partially explain spurious associations <cite class="ltx_cite ltx_citemacro_citep">(Astle and Balding, <a href="#bib.bib1" title="" class="ltx_ref">2009</a>)</cite>.
Environmental confounding factors can also be an obstacle for detecting G<math id="S1.p3.m2" class="ltx_Math" alttext="\times" display="inline"><mo>×</mo></math>E interactions. VanderWeele et al. have studied extensively the implications of confounding factors in G<math id="S1.p3.m3" class="ltx_Math" alttext="\times" display="inline"><mo>×</mo></math>E interactions studies <cite class="ltx_cite ltx_citemacro_citep">(VanderWeele et al., <a href="#bib.bib28" title="" class="ltx_ref">2013</a>, <a href="#bib.bib27" title="" class="ltx_ref">2012</a>)</cite>. However, they were more concerned about the rise of false positives in presence of environmental confounding than by the lose of power to detect the causal loci. Indeed, if the environmental factor that interacts with a susceptibility gene is unobserved but is correlated at some extent with one or several observed factors, not taking it into account could decrease radically the power to detect the G<math id="S1.p3.m4" class="ltx_Math" alttext="\times" display="inline"><mo>×</mo></math>E interactions.</p>
</div>
<div id="S1.p4" class="ltx_para">
<p class="ltx_p">Nowadays, several methods are available to perform GWAS. In principle, they could be used to detect G<math id="S1.p4.m1" class="ltx_Math" alttext="\times" display="inline"><mo>×</mo></math>E interactions. Among them, PLINK <cite class="ltx_cite ltx_citemacro_citep">(Purcell et al., <a href="#bib.bib19" title="" class="ltx_ref">2007</a>)</cite> can be considered as a gold standard for classical analysis. A major concern in GWAS is the need to account for the complicate dependence structure in the data, between loci as well as between individuals. Effects of population stratification can be easily accounted in PLINK by adding the PCA’s first components as covariates.</p>
</div>
<div id="S1.p5" class="ltx_para">
<p class="ltx_p">As an alternative, linear mixed models stem as promising statistical methods to correct for the stratification in the population. A popular implementation of linear mixed models is Fast-LMM <cite class="ltx_cite ltx_citemacro_citep">(Lippert et al., <a href="#bib.bib15" title="" class="ltx_ref">2011</a>)</cite>.</p>
</div>
<div id="S1.p6" class="ltx_para">
<p class="ltx_p">Furthermore, powerful data mining techniques are being increasingly used. Among them, the application of random forests (RFs) to the discovery of SNPs related to human diseases has grown in recent years <cite class="ltx_cite ltx_citemacro_citep">(Goldstein et al., <a href="#bib.bib10" title="" class="ltx_ref">2010</a>)</cite>.</p>
</div>
<div id="S1.p7" class="ltx_para">
<p class="ltx_p">Each time a new method is introduced, it is obviously essential to evaluate its performance in comparison with existing techniques through power studies <cite class="ltx_cite ltx_citemacro_citep">(Spencer et al., <a href="#bib.bib23" title="" class="ltx_ref">2009</a>)</cite>. However, such studies are often conducted by performing H1 simulations under models which are very similar to the ones used to design the new method, thus giving it an obvious advantage over the other methods. It is hence quite common to see many concurrent approaches each claiming to outperform all others.</p>
</div>
<div id="S1.p8" class="ltx_para">
<p class="ltx_p">Recently, a new method for H1 simulations called <math id="S1.p8.m1" class="ltx_Math" alttext="{\tt waffect}" display="inline"><mi>𝚠𝚊𝚏𝚏𝚎𝚌𝚝</mi></math> (pronounced ”double-u affect” for weighted affectations) has been introduced to avoid this issue <cite class="ltx_cite ltx_citemacro_citep">(Perduca et al., <a href="#bib.bib18" title="" class="ltx_ref">2012</a>)</cite>. Indeed, this method does not make any other assumption than the causal disease model itself, whose choice is completely unconstrained. <math id="S1.p8.m2" class="ltx_Math" alttext="{\tt waffect}" display="inline"><mi>𝚠𝚊𝚏𝚏𝚎𝚌𝚝</mi></math> uses weighted permutations to generate phenotypes conditionally to the genotypes and covariates by taking into account both the causal disease model and the design of the study. With this new approach, it is hence possible to produce non-subjective H1 datasets which do not favor one analysis method over the others.</p>
</div>
<div id="S1.p9" class="ltx_para">
<p class="ltx_p">The aim of this paper is twofold: 1- To propose an useful simulated dataset based on real genotypes with a confounding factor that interacts with the causal locus and is correlated with the observed covariates. For this purpose, we exploited the publicly available HapMap project datasets <cite class="ltx_cite ltx_citemacro_citep">(Thorisson et al., <a href="#bib.bib25" title="" class="ltx_ref">2005</a>)</cite> to obtain real genotypes with population structures, and we used <math id="S1.p9.m1" class="ltx_Math" alttext="{\tt waffect}" display="inline"><mi>𝚠𝚊𝚏𝚏𝚎𝚌𝚝</mi></math> to generate phenotypes for a chosen causal disease model. This simulated dataset could be very useful for future works aiming at studying the many issues involved in the detection of G<math id="S1.p9.m2" class="ltx_Math" alttext="\times" display="inline"><mo>×</mo></math>E interactions in GWAS.
2- To study the effect of a confounding factor on the power to detect G<math id="S1.p9.m3" class="ltx_Math" alttext="\times" display="inline"><mo>×</mo></math>E interactions in GWAS. For this purpose, we compared four approaches based on three popular methods (PLINK, Fast-LMM and random forests) by performing power analysis on our simulated dataset.</p>
</div>
</section>
<section id="S2" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">2 </span>Simulated Dataset</h2>

<section id="S2.SS1" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">2.1 </span>Genotypes</h3>

<div id="S2.SS1.p1" class="ltx_para">
<p class="ltx_p">The genotypic dataset used in our study was extracted from the HapMap phase III database of genetic variations <cite class="ltx_cite ltx_citemacro_citep">(Gibbs et al., <a href="#bib.bib9" title="" class="ltx_ref">2003</a>)</cite>. This database investigates 11 human populations including 57 unrelated MEX, 146 unrelated YRI, 52 unrelated ASW, 110 unrelated CEU, 154 unrelated MKK, 137 unrelated CHB, 109 unrelated CHD, 101 unrelated GIH, 113 unrelated JPT, 110 unrelated LWK and 102 unrelated TSI. Only the SNPs that are shared by all populations were retained.</p>
</div>
<div id="S2.SS1.p2" class="ltx_para">
<p class="ltx_p">Principal component analysis (PCA) was performed on the whole genome, keeping one SNP over 1000 SNPs; the first five principal components (<math id="S2.SS1.p2.m1" class="ltx_Math" alttext="{\tt pca}_{\,i},i=1,\ldots,5" display="inline"><mrow><mrow><mrow><msub><mi>𝚙𝚌𝚊</mi><mi>i</mi></msub><mo>,</mo><mi>i</mi></mrow><mo>=</mo><mn>1</mn></mrow><mo>,</mo><mrow><mi mathvariant="normal">…</mi><mo>,</mo><mn>5</mn></mrow></mrow></math>) were considered as covariates. Figure <a href="#S5.F1" title="Figure 1 ‣ Non-subjective power analysis to detect G×E interactions in Genome-Wide Association Studies in presence of confounding factor" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1</span></a> shows that the PCs allow to retrieve easily the information about the population.</p>
</div>
<div id="S2.SS1.p3" class="ltx_para">
<p class="ltx_p">The association analysis in our study were conducted on Chromosome 6, after quality control including Hardy Weinberg equilibrium testing and exclusion of SNPs with a minor allele frequency (MAF) less than 5%.</p>
</div>
</section>
<section id="S2.SS2" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">2.2 </span>Covariates</h3>

<div id="S2.SS2.p1" class="ltx_para">
<p class="ltx_p">Covariates and phenotypes were simulated in order to mimic a complex interaction between an arbitrarily chosen causal SNP and a confounding factor through a latent exposure (called <math id="S2.SS2.p1.m1" class="ltx_Math" alttext="{\tt treatment}" display="inline"><mi>𝚝𝚛𝚎𝚊𝚝𝚖𝚎𝚗𝚝</mi></math>). The latent exposure was defined with high correlation with two observed covariates (<math id="S2.SS2.p1.m2" class="ltx_Math" alttext="{\tt bmi}" display="inline"><mi>𝚋𝚖𝚒</mi></math>, for body mass index, and <math id="S2.SS2.p1.m3" class="ltx_Math" alttext="{\tt sex}" display="inline"><mi>𝚜𝚎𝚡</mi></math>) as well as with the population of belonging. The idea is that the treatment is typically taken by women (and less often by men) trying to loose weight.</p>
</div>
<div id="S2.SS2.p2" class="ltx_para">
<p class="ltx_p"><math id="S2.SS2.p2.m1" class="ltx_Math" alttext="\tt{bmi}" display="inline"><mi>𝚋𝚖𝚒</mi></math> was simulated taken into account the five first principal components and another environmental covariate denoted <math id="S2.SS2.p2.m2" class="ltx_Math" alttext="{\tt smoking}" display="inline"><mi>𝚜𝚖𝚘𝚔𝚒𝚗𝚐</mi></math>. This binary variable was simulated to mimic smoking behaviors with a probability distribution which depends on the population and sex. Indeed, women usually smoke less than men with this difference depending on the population. In order to simulate the smoking covariate, the eleven populations were classified in three sub-populations: European (E); African (Af) and Asian (As).</p>
</div>
<div id="S2.SS2.p3" class="ltx_para">
<p class="ltx_p">In the European sub-population, we considered that 32% of individuals were smokers. More specifically, we supposed that 37% of men and 27% of women were smokers. In the African sub-population, the prevalence of smoking was supposed to be 27%: 43.8% among men and 12.9% among women <cite class="ltx_cite ltx_citemacro_citep">(Christopoulou and Lillard, <a href="#bib.bib5" title="" class="ltx_ref">2011</a>)</cite>. At last, in the Asian sub-population, we considered that 27% of individuals were smokers: 45.7% among men and 4.8 % among women <cite class="ltx_cite ltx_citemacro_citep">(Tsai et al., <a href="#bib.bib26" title="" class="ltx_ref">2008</a>)</cite>. Covariate <math id="S2.SS2.p3.m1" class="ltx_Math" alttext="{\tt sex}" display="inline"><mi>𝚜𝚎𝚡</mi></math> was obtained from HapMap data.</p>
</div>
<div id="S2.SS2.p4" class="ltx_para">
<p class="ltx_p">Specifically, <math id="S2.SS2.p4.m1" class="ltx_Math" alttext="{\tt bmi}" display="inline"><mi>𝚋𝚖𝚒</mi></math> was simulated with a regression on the first five principal components in order to have 60% of heritability and with a residual standard deviation of <math id="S2.SS2.p4.m2" class="ltx_Math" alttext="4.0" display="inline"><mn>4.0</mn></math>. To take into account the fact that in average smokers have a lower <math id="S2.SS2.p4.m3" class="ltx_Math" alttext="{\tt bmi}" display="inline"><mi>𝚋𝚖𝚒</mi></math> than non-smokers, we simulated a smoker effect in the <math id="S2.SS2.p4.m4" class="ltx_Math" alttext="{\tt bmi}" display="inline"><mi>𝚋𝚖𝚒</mi></math> covariate by adding a score of 1.5 to the <span class="ltx_text ltx_font_typewriter">bmi</span> average for non-smokers <cite class="ltx_cite ltx_citemacro_citep">(Chiolero et al., <a href="#bib.bib4" title="" class="ltx_ref">2008</a>)</cite>.</p>
</div>
<div id="S2.SS2.p5" class="ltx_para">
<p class="ltx_p">The individual probability to take a treatment, <math id="S2.SS2.p5.m1" class="ltx_Math" alttext="\mathbb{P}({\tt treatment})" display="inline"><mrow><mi>ℙ</mi><mo>⁢</mo><mrow><mo stretchy="false">(</mo><mi>𝚝𝚛𝚎𝚊𝚝𝚖𝚎𝚗𝚝</mi><mo stretchy="false">)</mo></mrow></mrow></math>, was correlated with covariates <math id="S2.SS2.p5.m2" class="ltx_Math" alttext="{\tt bmi}" display="inline"><mi>𝚋𝚖𝚒</mi></math>, <math id="S2.SS2.p5.m3" class="ltx_Math" alttext="{\tt sex}" display="inline"><mi>𝚜𝚎𝚡</mi></math> and the population of belonging as follows:</p>
</div>
<div id="S2.SS2.p6" class="ltx_para">
<table id="S2.Ex1" class="ltx_equation ltx_eqn_table">

<tbody><tr class="ltx_equation ltx_eqn_row ltx_align_baseline">
<td class="ltx_eqn_cell ltx_eqn_center_padleft"></td>
<td class="ltx_eqn_cell ltx_align_center"><math id="S2.Ex1.m1" class="ltx_Math" alttext="1/\mathbb{P}({\tt treatment})=(1+2\times 1_{{\tt sex}=1})\times[1+\exp({-{\tt
bmi%
}+25+\gamma})]" display="block"><mrow><mrow><mrow><mn>1</mn><mo>/</mo><mi>ℙ</mi></mrow><mo>⁢</mo><mrow><mo stretchy="false">(</mo><mi>𝚝𝚛𝚎𝚊𝚝𝚖𝚎𝚗𝚝</mi><mo stretchy="false">)</mo></mrow></mrow><mo>=</mo><mrow><mrow><mo stretchy="false">(</mo><mrow><mn>1</mn><mo>+</mo><mrow><mn>2</mn><mo lspace="0.222em" rspace="0.222em">×</mo><msub><mn>1</mn><mrow><mi>𝚜𝚎𝚡</mi><mo>=</mo><mn>1</mn></mrow></msub></mrow></mrow><mo rspace="0.055em" stretchy="false">)</mo></mrow><mo rspace="0.222em">×</mo><mrow><mo stretchy="false">[</mo><mrow><mn>1</mn><mo>+</mo><mrow><mi>exp</mi><mo>⁡</mo><mrow><mo stretchy="false">(</mo><mrow><mrow><mo>−</mo><mi>𝚋𝚖𝚒</mi></mrow><mo>+</mo><mn>25</mn><mo>+</mo><mi>γ</mi></mrow><mo stretchy="false">)</mo></mrow></mrow></mrow><mo stretchy="false">]</mo></mrow></mrow></mrow></math></td>
<td class="ltx_eqn_cell ltx_eqn_center_padright"></td>
</tr></tbody>
</table>
<p class="ltx_p">where <math id="S2.SS2.p6.m1" class="ltx_Math" alttext="\gamma\in\{-\inf,-0.1,0,0.15,-0.45,0.35,0.6,-0.4,0.05,0.1\}" display="inline"><mrow><mi>γ</mi><mo>∈</mo><mrow><mo stretchy="false">{</mo><mrow><mo>−</mo><mo lspace="0.167em" rspace="0em">inf</mo></mrow><mo>,</mo><mrow><mo>−</mo><mn>0.1</mn></mrow><mo>,</mo><mn>0</mn><mo>,</mo><mn>0.15</mn><mo>,</mo><mrow><mo>−</mo><mn>0.45</mn></mrow><mo>,</mo><mn>0.35</mn><mo>,</mo><mn>0.6</mn><mo>,</mo><mrow><mo>−</mo><mn>0.4</mn></mrow><mo>,</mo><mn>0.05</mn><mo>,</mo><mn>0.1</mn><mo stretchy="false">}</mo></mrow></mrow></math> for population 1 to 11.</p>
</div>
<div id="S2.SS2.p7" class="ltx_para">
<p class="ltx_p">To sum up, in our standard design covariates <math id="S2.SS2.p7.m1" class="ltx_Math" alttext="{\tt sex}" display="inline"><mi>𝚜𝚎𝚡</mi></math>, <math id="S2.SS2.p7.m2" class="ltx_Math" alttext="{\tt pca}_{\,i},(i=1,\ldots,5)" display="inline"><mrow><msub><mi>𝚙𝚌𝚊</mi><mi>i</mi></msub><mo>,</mo><mrow><mo stretchy="false">(</mo><mrow><mi>i</mi><mo>=</mo><mrow><mn>1</mn><mo>,</mo><mi mathvariant="normal">…</mi><mo>,</mo><mn>5</mn></mrow></mrow><mo stretchy="false">)</mo></mrow></mrow></math>, <math id="S2.SS2.p7.m3" class="ltx_Math" alttext="{\tt bmi}" display="inline"><mi>𝚋𝚖𝚒</mi></math> and <math id="S2.SS2.p7.m4" class="ltx_Math" alttext="{\tt smoking}" display="inline"><mi>𝚜𝚖𝚘𝚔𝚒𝚗𝚐</mi></math> were supposed observed. The population of belonging, covariate <math id="S2.SS2.p7.m5" class="ltx_Math" alttext="{\tt treatment}" display="inline"><mi>𝚝𝚛𝚎𝚊𝚝𝚖𝚎𝚗𝚝</mi></math> and the other principal components were supposed unknown (even though these are easily computable).</p>
</div>
</section>
<section id="S2.SS3" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">2.3 </span>Disease Model</h3>

<div id="S2.SS3.p1" class="ltx_para">
<p class="ltx_p">We arbitrarily chose the SNP in position 22,683,075 in a dense area of chromosome 6, as the binary susceptibility locus, denoted <math id="S2.SS3.p1.m1" class="ltx_Math" alttext="{\tt causalSNP}" display="inline"><mi>𝚌𝚊𝚞𝚜𝚊𝚕𝚂𝙽𝙿</mi></math>. Assuming a dominant effect, we encoded <math id="S2.SS3.p1.m2" class="ltx_Math" alttext="{\tt causalSNP}=1" display="inline"><mrow><mi>𝚌𝚊𝚞𝚜𝚊𝚕𝚂𝙽𝙿</mi><mo>=</mo><mn>1</mn></mrow></math> in presence of at least one minor frequency allele and <math id="S2.SS3.p1.m3" class="ltx_Math" alttext="{\tt causalSNP}=0" display="inline"><mrow><mi>𝚌𝚊𝚞𝚜𝚊𝚕𝚂𝙽𝙿</mi><mo>=</mo><mn>0</mn></mrow></math> otherwise. We considered a disease model with a very strong G<math id="S2.SS3.p1.m4" class="ltx_Math" alttext="\times" display="inline"><mo>×</mo></math>E interaction (relative risk of <math id="S2.SS3.p1.m5" class="ltx_Math" alttext="50" display="inline"><mn>50</mn></math>) with a baseline prevalence of <math id="S2.SS3.p1.m6" class="ltx_Math" alttext="1\%" display="inline"><mrow><mn>1</mn><mo>%</mo></mrow></math>:</p>
<table id="S2.Ex2" class="ltx_equation ltx_eqn_table">

<tbody><tr class="ltx_equation ltx_eqn_row ltx_align_baseline">
<td class="ltx_eqn_cell ltx_eqn_center_padleft"></td>
<td class="ltx_eqn_cell ltx_align_center"><math id="S2.Ex2.m1" class="ltx_Math" alttext="\mathbb{P}({\tt disease})=0.01\times(1.0+50.0\times 1_{{\tt causalSNP}=1}%
\times 1_{{\tt treatment}=1})" display="block"><mrow><mrow><mi>ℙ</mi><mo>⁢</mo><mrow><mo stretchy="false">(</mo><mi>𝚍𝚒𝚜𝚎𝚊𝚜𝚎</mi><mo stretchy="false">)</mo></mrow></mrow><mo>=</mo><mrow><mn>0.01</mn><mo lspace="0.222em" rspace="0.222em">×</mo><mrow><mo stretchy="false">(</mo><mrow><mn>1.0</mn><mo>+</mo><mrow><mn>50.0</mn><mo lspace="0.222em" rspace="0.222em">×</mo><msub><mn>1</mn><mrow><mi>𝚌𝚊𝚞𝚜𝚊𝚕𝚂𝙽𝙿</mi><mo>=</mo><mn>1</mn></mrow></msub><mo lspace="0.222em" rspace="0.222em">×</mo><msub><mn>1</mn><mrow><mi>𝚝𝚛𝚎𝚊𝚝𝚖𝚎𝚗𝚝</mi><mo>=</mo><mn>1</mn></mrow></msub></mrow></mrow><mo stretchy="false">)</mo></mrow></mrow></mrow></math></td>
<td class="ltx_eqn_cell ltx_eqn_center_padright"></td>
</tr></tbody>
</table>
</div>
<div id="S2.SS3.p2" class="ltx_para">
<p class="ltx_p">It is important to stress that this is not a very realistic model of complex disease due to the lack of genetic marginal effects and also due to the very strong interaction with a relative risk (RR) equal to 50. We chose not to include any marginal effects for sake of clarity and because we were interested on the detection of G<math id="S2.SS3.p2.m1" class="ltx_Math" alttext="\times" display="inline"><mo>×</mo></math>E interactions. Concerning the strong G<math id="S2.SS3.p2.m2" class="ltx_Math" alttext="\times" display="inline"><mo>×</mo></math>E interaction, we chose to have a relative risk as 50 in order to increase the chance of detecting the G<math id="S2.SS3.p2.m3" class="ltx_Math" alttext="\times" display="inline"><mo>×</mo></math>E interaction in presence of the simulated confounding factor.</p>
</div>
<div id="S2.SS3.p3" class="ltx_para">
<p class="ltx_p">Phenotypes were simulated accordingly to the disease models by means of the package <math id="S2.SS3.p3.m1" class="ltx_Math" alttext="{\tt waffect}" display="inline"><mi>𝚠𝚊𝚏𝚏𝚎𝚌𝚝</mi></math> <cite class="ltx_cite ltx_citemacro_citep">(Perduca et al., <a href="#bib.bib18" title="" class="ltx_ref">2012</a>)</cite> publicly available on the CRAN server of R packages <cite class="ltx_cite ltx_citemacro_citep">(R Core Team, <a href="#bib.bib20" title="" class="ltx_ref">2013</a>)</cite>. This enabled us to simulate exactly 595 cases and 596 controls for the 1191 individuals from the HapMap genotypic dataset (see next session for a comprehensive introduction to <math id="S2.SS3.p3.m2" class="ltx_Math" alttext="{\tt waffect}" display="inline"><mi>𝚠𝚊𝚏𝚏𝚎𝚌𝚝</mi></math> ).</p>
</div>
</section>
</section>
<section id="S3" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">3 </span>Power analysis</h2>

<section id="S3.SS1" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">3.1 </span>Phenotype Simulations</h3>

<div id="S3.SS1.p1" class="ltx_para">
<p class="ltx_p">In order to assess the empirical statistical power of different tools to detect associations, we simulated 200 phenotypes replicates under the disease model H1 and 200 phenotypes replicates under the null hypothesis H0 of no association. Each replicate consists of 1191 phenotypes, one for each individual.</p>
</div>
<div id="S3.SS1.p2" class="ltx_para">
<p class="ltx_p">The simulations under H0 were obtained by simply permuting the phenotypes, thus breaking potential associations between phenotypes and genotypes. The simulations under the alternative hypothesis H1 were performed using the R package <math id="S3.SS1.p2.m1" class="ltx_Math" alttext="{\tt waffect}" display="inline"><mi>𝚠𝚊𝚏𝚏𝚎𝚌𝚝</mi></math> publicly available on CRAN <cite class="ltx_cite ltx_citemacro_citep">(Perduca et al., <a href="#bib.bib18" title="" class="ltx_ref">2012</a>)</cite>.</p>
</div>
<div id="S3.SS1.p3" class="ltx_para">
<p class="ltx_p">The principal function in <math id="S3.SS1.p3.m1" class="ltx_Math" alttext="{\tt waffect}" display="inline"><mi>𝚠𝚊𝚏𝚏𝚎𝚌𝚝</mi></math> is based on a backward sampling algorithm which makes it possible to generate weighted permutations. For the purposes of phenotype simulation, the vector of weights is given by the penetrance, that is the probability for each individual to be a case according to the disease model. One crucial consequence of using weighted permutations is that the number of cases and controls is constant across the replicates. This makes it possible to respect the original design in each replicate and therefore to compare the performance of an association method across different replicates.</p>
</div>
<div id="S3.SS1.p4" class="ltx_para">
<p class="ltx_p">Simulating phenotypes rather than genotypes, as does the gold standard Hapgen <cite class="ltx_cite ltx_citemacro_citep">(Su et al., <a href="#bib.bib24" title="" class="ltx_ref">2011</a>)</cite>, does not require additional data such as haplotype frequencies or recombination rates and has the obvious advantages of requiring much smaller computational memory and time. Moreover, for the purposes of the present work, the primary benefit of using <math id="S3.SS1.p4.m1" class="ltx_Math" alttext="{\tt waffect}" display="inline"><mi>𝚠𝚊𝚏𝚏𝚎𝚌𝚝</mi></math> is that it only requires a vector of probabilities as input. As a result, the choice of the disease model is totally unconstrained; in particular it is possible to include G<math id="S3.SS1.p4.m2" class="ltx_Math" alttext="\times" display="inline"><mo>×</mo></math>E interactions.</p>
</div>
<div id="S3.SS1.p5" class="ltx_para">
<p class="ltx_p">In principle, one could achieve the same result by simply using a rejection algorithm which samples the phenotype of each individual according to the probability to be a case and then accepts the resulting replicate only if there are <em class="ltx_emph ltx_font_italic">enough</em> cases. Because the probability of obtaining a full configuration of phenotypes with the correct number of cases is extremely low, this approach cannot be used in practice. In order to overcome this problem, a solution often applied in practice is to increase the prevalence in the disease model, maintaining unchanged the relative risks. However it can be proved that adjusting the prevalence creates bias in the empirical power estimate <cite class="ltx_cite ltx_citemacro_citep">(Perduca et al., <a href="#bib.bib18" title="" class="ltx_ref">2012</a>)</cite>.</p>
</div>
</section>
<section id="S3.SS2" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">3.2 </span>Statistical analysis</h3>

<div id="S3.SS2.p1" class="ltx_para">
<p class="ltx_p">In this section, we briefly describe the four popular approaches adopted in our study to perform GWA analysis. The goal standard PLINK <cite class="ltx_cite ltx_citemacro_citep">(individual SNP logistic regression, Purcell et al., <a href="#bib.bib19" title="" class="ltx_ref">2007</a>)</cite> was applied in two alternative approaches : 1- analysis performed regardless of G<math id="S3.SS2.p1.m1" class="ltx_Math" alttext="\times" display="inline"><mo>×</mo></math>E interactions and considering only genetic effects; and, since it is easy to consider interaction terms with PLINK, 2- analysis performed taking into account G<math id="S3.SS2.p1.m2" class="ltx_Math" alttext="\times" display="inline"><mo>×</mo></math>E interactions. The other two approaches are 3- the linear mixed model for population structure correction implemented in Fast-LMM <cite class="ltx_cite ltx_citemacro_citep">(Lippert et al., <a href="#bib.bib15" title="" class="ltx_ref">2011</a>)</cite>, and 4- the random forest data mining technique <cite class="ltx_cite ltx_citemacro_citep">(RandomForest R package, Liaw and Wiener, <a href="#bib.bib14" title="" class="ltx_ref">2002</a>)</cite>.</p>
</div>
<section id="S3.SS2.SSS1" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">3.2.1 </span>PLINK</h4>

<div id="S3.SS2.SSS1.p1" class="ltx_para">
<p class="ltx_p">PLINK implements a logistic regression approach <cite class="ltx_cite ltx_citemacro_citep">(Purcell et al., <a href="#bib.bib19" title="" class="ltx_ref">2007</a>)</cite> that allows for multiple binary or continuous covariates when testing for disease trait SNP association and interactions with covariates.
PLINK provides p-values for significance coefficients in the logistic model. In this work, we considered two approaches using PLINK.</p>
</div>
<div id="S3.SS2.SSS1.p2" class="ltx_para">
<p class="ltx_p">The first approach, which we referred to as ”PLINK SNP”, consisted in performing analysis regardless of G<math id="S3.SS2.SSS1.p2.m1" class="ltx_Math" alttext="\times" display="inline"><mo>×</mo></math>E interactions by looking at the p-value associated to the significant coefficients for the SNPs.</p>
</div>
<div id="S3.SS2.SSS1.p3" class="ltx_para">
<p class="ltx_p">We referred to the second approach as ”PLINK SNP<math id="S3.SS2.SSS1.p3.m1" class="ltx_Math" alttext="\times" display="inline"><mo>×</mo></math>COV”, where COV is the environmental covariate under consideration (either <math id="S3.SS2.SSS1.p3.m2" class="ltx_Math" alttext="\tt{bmi}" display="inline"><mi>𝚋𝚖𝚒</mi></math> in our standard design or <math id="S3.SS2.SSS1.p3.m3" class="ltx_Math" alttext="\tt{treatment}" display="inline"><mi>𝚝𝚛𝚎𝚊𝚝𝚖𝚎𝚗𝚝</mi></math> in our further analysis, see below). PLINK SNP<math id="S3.SS2.SSS1.p3.m4" class="ltx_Math" alttext="\times" display="inline"><mo>×</mo></math>COV accounted for all the interactions between the analyzed SNPs and the environmental factor considered through the p-values associated to the significance coefficients of such interactions.</p>
</div>
<div id="S3.SS2.SSS1.p4" class="ltx_para">
<p class="ltx_p">Correction for population structure was taken into account by considering the five first principal components resulting from the PCA performed on the whole genome.</p>
</div>
</section>
<section id="S3.SS2.SSS2" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">3.2.2 </span>Random Forests</h4>

<div id="S3.SS2.SSS2.p1" class="ltx_para">
<p class="ltx_p">Random forests (RFs) have been introduced by <cite class="ltx_cite ltx_citemacro_citet">Breiman (<a href="#bib.bib3" title="" class="ltx_ref">2001</a>)</cite>. The general principle consists in building repeatedly classification and regression trees (CART) from bootstrapping of the original data. This process produces a forest of classification trees which are statistically analyzed to produce importance measures of the covariates (e.g. a variable belonging to many trees probably plays a key role in the classification).</p>
</div>
<div id="S3.SS2.SSS2.p2" class="ltx_para">
<p class="ltx_p">Random forests are a popular way to perform data mining on GWAS data. Despite the fact that they exploit heavily marginal linear regressions, random forests are able to detect interactions between variables <cite class="ltx_cite ltx_citemacro_citep">(see Boulesteix et al., <a href="#bib.bib2" title="" class="ltx_ref">2012</a>, for an overview of random forests in the GWAS context)</cite>. Recently, a regularized version of random forests has been proposed to deal with high dimensional data <cite class="ltx_cite ltx_citemacro_citep">(Deng and Runger, <a href="#bib.bib7" title="" class="ltx_ref">2012</a>)</cite>. In this work we decided to disregard this approach because it was too slow on our data to be practical.</p>
</div>
<div id="S3.SS2.SSS2.p3" class="ltx_para">
<p class="ltx_p">For our random forests analysis, we used the <span class="ltx_text ltx_font_typewriter">randomForest</span> package (version 4.6-7) from R <cite class="ltx_cite ltx_citemacro_citep">(R Core Team, <a href="#bib.bib20" title="" class="ltx_ref">2013</a>)</cite>. We simply used the default parameters of the method with the disease status as a binary outcome and with all observed covariates and SNPs as explanatory variables. Once the forest computed, we extracted for each variable its importance measure using the default approach of the package (normalized difference between out-of-bag proportion error using original data or a permuted version). We hence obtained for each variable and each replicate a real value which reflects the importance of the variable for discriminating between cases and controls. The higher this importance value, the stronger the association with the disease.</p>
</div>
</section>
<section id="S3.SS2.SSS3" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">3.2.3 </span>Fast-LMM</h4>

<div id="S3.SS2.SSS3.p1" class="ltx_para">
<p class="ltx_p">It is a well known problem that in GWAS confounding effects of population structure lead to false positive and therefore need to be taken into account. An alternative to including the first principal components in linear or logistic regression models in order to correct for confounding factors are Linear Mixed Models <cite class="ltx_cite ltx_citemacro_citep">(LMMs, Hoffman, <a href="#bib.bib12" title="" class="ltx_ref">2013</a>)</cite>. LMMs generalize linear models by introducing random effects as predictors, in addition to the usual fixed effects. Indeed, LMMs are known to be effective when observations are not independent but rather involve related individuals.</p>
</div>
<div id="S3.SS2.SSS3.p2" class="ltx_para">
<p class="ltx_p">In LMMs for GWAS, the random effect is expressed by a multivariate normal distribution whose variance-covariance matrix measures the genetic similarity between individuals. Recently the algorithm Fast-LMM has been introduced to perform efficiently exact inference for LMMs <cite class="ltx_cite ltx_citemacro_citep">(Lippert et al., <a href="#bib.bib15" title="" class="ltx_ref">2011</a>)</cite>. Roughly speaking, Fast-LMM (for Factored spectrally transformed Linear Mixed Models) is based on a spectral decomposition of the genetic similarity matrix which rotates the phenotypes into uncorrelated phenotypes thus converting the original estimation problem into the maximization of the likelihood of a linear regression model.</p>
</div>
<div id="S3.SS2.SSS3.p3" class="ltx_para">
<p class="ltx_p">A drawback of the current implementation of Fast-LMM is that it does not allow to consider explicit interaction terms between genotypic variables and covariates in the linear mixed model <span id="footnote1" class="ltx_note ltx_role_footnote"><sup class="ltx_note_mark">1</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">1</sup>
                <span class="ltx_tag ltx_tag_note">1</span>
                
                
                
              (We unsuccessfully tried to contact the authors of Fast-LMM on this matter.)</span></span></span>. A possible solution to overcome this limitation is to code directly such interactions in the covariate file, thus adding new columns. However, this solution was not appropriate in this context because it would have required to magnify several times the size of the variables file in order to consider the cartesian product of all the SNPs with all the covariates. We then we simply applied Fast-LMM to the original genotypic and covariate datasets.</p>
</div>
</section>
</section>
<section id="S3.SS3" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">3.3 </span>Power, ROC curves and AUC</h3>

<div id="S3.SS3.p1" class="ltx_para">
<p class="ltx_p">In order to evaluate empirically the performance of the methods described above, we computed four summary statistics, one for each method. These global statistics were then used to estimate the Area under the Curve (AUC) corresponding to the receiver operating characteristic (ROC) curves of the four methods, each expressing the performance of the corresponding method.</p>
</div>
<div id="S3.SS3.p2" class="ltx_para">
<p class="ltx_p">More specifically, for PLINK SNP we took as simple global statistics the smallest among all the p-values associated to the significance coefficients of the SNPs, similarly for Fast-LMM. For PLINK SNP<math id="S3.SS3.p2.m1" class="ltx_Math" alttext="\times" display="inline"><mo>×</mo></math>COV, we took the smallest among all the p-values associated to significance coefficients of the terms coding for the interactions between the SNPs and the covariate (<span class="ltx_text ltx_font_typewriter">bmi</span> or <span class="ltx_text ltx_font_typewriter">treatment</span>). At last, the summary statistics for the random forest-based method was defined as the maximum of the importance statistics over all considered SNPs.</p>
</div>
<div id="S3.SS3.p3" class="ltx_para">
<p class="ltx_p">Then, for each method, we obtained two vectors of length 200, one under H0 and one under H1. These two vectors of comprehensive signals were used to estimate the ROC curves of the four methods using the R package pROC <cite class="ltx_cite ltx_citemacro_citep">(Robin et al., <a href="#bib.bib21" title="" class="ltx_ref">2011</a>)</cite>.</p>
</div>
<div id="S3.SS3.p4" class="ltx_para">
<p class="ltx_p">We recall that ROC curves provide a graphical representation of the specificities and sensitivities (i.e. values of statistical power) that can be obtained for all possible values of the threshold of significance <cite class="ltx_cite ltx_citemacro_citep">(Metz, <a href="#bib.bib16" title="" class="ltx_ref">1978</a>)</cite>. An informative summary of the ROC curve information is the area under the ROC curve (i.e. AUC). The AUC can be qualitatively interpreted as follows: <math id="S3.SS3.p4.m1" class="ltx_Math" alttext="\text{AUC}\leq 0.6" display="inline"><mrow><mtext>AUC</mtext><mo>≤</mo><mn>0.6</mn></mrow></math> means ‘fail’; <math id="S3.SS3.p4.m2" class="ltx_Math" alttext="0.6&lt;\text{AUC}\leq 0.70" display="inline"><mrow><mn>0.6</mn><mo>&lt;</mo><mtext>AUC</mtext><mo>≤</mo><mn>0.70</mn></mrow></math> means ‘poor’; <math id="S3.SS3.p4.m3" class="ltx_Math" alttext="0.7&lt;\text{AUC}\leq 0.80" display="inline"><mrow><mn>0.7</mn><mo>&lt;</mo><mtext>AUC</mtext><mo>≤</mo><mn>0.80</mn></mrow></math> means ‘fair’; <math id="S3.SS3.p4.m4" class="ltx_Math" alttext="0.8&lt;\text{AUC}\leq 0.9" display="inline"><mrow><mn>0.8</mn><mo>&lt;</mo><mtext>AUC</mtext><mo>≤</mo><mn>0.9</mn></mrow></math> means ‘good’; <math id="S3.SS3.p4.m5" class="ltx_Math" alttext="0.9&lt;\text{AUC}\leq 1.0" display="inline"><mrow><mn>0.9</mn><mo>&lt;</mo><mtext>AUC</mtext><mo>≤</mo><mn>1.0</mn></mrow></math> means ‘excellent’.</p>
</div>
</section>
</section>
<section id="S4" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">4 </span>Results and discussion</h2>

<div id="S4.p1" class="ltx_para">
<p class="ltx_p">Association analysis were adjusted on covariates <math id="S4.p1.m1" class="ltx_Math" alttext="\tt{sex}" display="inline"><mi>𝚜𝚎𝚡</mi></math>, <math id="S4.p1.m2" class="ltx_Math" alttext="\tt{smoking}" display="inline"><mi>𝚜𝚖𝚘𝚔𝚒𝚗𝚐</mi></math> together with either <span class="ltx_text ltx_font_typewriter">bmi</span> (our standard design) or <span class="ltx_text ltx_font_typewriter">treatment</span>. Moreover, for PLINK and random forests the five first principal components <math id="S4.p1.m3" class="ltx_Math" alttext="\tt{pc_{i},i\in\{1,\dots,5\}}" display="inline"><mrow><mrow><msub><mi>𝚙𝚌</mi><mi>𝚒</mi></msub><mo>,</mo><mi>𝚒</mi></mrow><mo>∈</mo><mrow><mo stretchy="false">{</mo><mn>𝟷</mn><mo>,</mo><mi mathvariant="normal">…</mi><mo>,</mo><mn>𝟻</mn><mo stretchy="false">}</mo></mrow></mrow></math> were included as predictors. We recall that in order to evaluate empirically the detection power of the four approaches in presence of confounding factor, analysis were performed on 200+200 phenotypic replicates under H0 and H1. Investigations were performed on chromosome 6 and subregions.</p>
</div>
<section id="S4.SS1" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">4.1 </span>Confounding factor is not known</h3>

<div id="S4.SS1.p1" class="ltx_para">
<p class="ltx_p">At first, the latent exposure <math id="S4.SS1.p1.m1" class="ltx_Math" alttext="\tt{treatment}" display="inline"><mi>𝚝𝚛𝚎𝚊𝚝𝚖𝚎𝚗𝚝</mi></math> (i.e. the confounding factor) was supposed to be unknown : the covariate <math id="S4.SS1.p1.m2" class="ltx_Math" alttext="\tt{bmi}" display="inline"><mi>𝚋𝚖𝚒</mi></math> was observed instead of <math id="S4.SS1.p1.m3" class="ltx_Math" alttext="\tt{treatment}" display="inline"><mi>𝚝𝚛𝚎𝚊𝚝𝚖𝚎𝚗𝚝</mi></math> and considered, mistakenly, as the environmental exposure of interest.</p>
</div>
<div id="S4.SS1.p2" class="ltx_para">
<p class="ltx_p">Table <a href="#S5.T1" title="Table 1 ‣ Non-subjective power analysis to detect G×E interactions in Genome-Wide Association Studies in presence of confounding factor" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1</span></a> shows the estimated AUC together with 95% confidence intervals for the four approaches. Obviously, the performance of each method increases when the region under consideration decreases and reach a good power when the region is restricted to the causal SNP. However power is low (fail or poor) when estimation are done on whole chromosome 6.
PLINK SNP, Fast-LMM and RF provide comparable results even if PLINK SNP seems to be a little more efficient.</p>
</div>
<div id="S4.SS1.p3" class="ltx_para">
<p class="ltx_p">Surprisingly, the estimated performance is better with PLINK SNP than with PLINK SNP<math id="S4.SS1.p3.m1" class="ltx_Math" alttext="\times\tt{bmi}" display="inline"><mrow><mi></mi><mo lspace="0.222em" rspace="0.222em">×</mo><mi>𝚋𝚖𝚒</mi></mrow></math>. For example, applying PLINK SNP on a region with less than 800 SNPs around the causal SNP provides good to excellent power while analysis with PLINK SNP<math id="S4.SS1.p3.m2" class="ltx_Math" alttext="\times\tt{bmi}" display="inline"><mrow><mi></mi><mo lspace="0.222em" rspace="0.222em">×</mo><mi>𝚋𝚖𝚒</mi></mrow></math> need to be restricted to the causal SNP to reach similar power. At first, this counter-intuitive result led us to believe that it is unnecessary to account for the G<math id="S4.SS1.p3.m3" class="ltx_Math" alttext="\times" display="inline"><mo>×</mo></math>E interactions in analysis since PLINK SNP provides better power than PLINK SNP<math id="S4.SS1.p3.m4" class="ltx_Math" alttext="\times\tt{bmi}" display="inline"><mrow><mi></mi><mo lspace="0.222em" rspace="0.222em">×</mo><mi>𝚋𝚖𝚒</mi></mrow></math>.</p>
</div>
<div id="S4.SS1.p4" class="ltx_para">
<p class="ltx_p">Furthermore, we note that AUC estimate applying PLINK SNP<math id="S4.SS1.p4.m1" class="ltx_Math" alttext="\times\tt{bmi}" display="inline"><mrow><mi></mi><mo lspace="0.222em" rspace="0.222em">×</mo><mi>𝚋𝚖𝚒</mi></mrow></math> when the analysis is performed on the whole chromosome 6 appears higher than on a region of 4000 SNPs around the causal SNP. This result could be explained by sample variability.</p>
</div>
</section>
<section id="S4.SS2" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">4.2 </span>Confounding factor is known</h3>

<div id="S4.SS2.p1" class="ltx_para">
<p class="ltx_p">To try to understand the fact that taking into account G<math id="S4.SS2.p1.m1" class="ltx_Math" alttext="\times" display="inline"><mo>×</mo></math>E interactions in presence of a confounding factor seems not to improve the performance, we performed association analysis by replacing the covariate <math id="S4.SS2.p1.m2" class="ltx_Math" alttext="\tt{bmi}" display="inline"><mi>𝚋𝚖𝚒</mi></math> by the confounding factor <math id="S4.SS2.p1.m3" class="ltx_Math" alttext="\tt{treatment}" display="inline"><mi>𝚝𝚛𝚎𝚊𝚝𝚖𝚎𝚗𝚝</mi></math>. This is possible because analysis are done on our simulated dataset for which we know the covariate <math id="S4.SS2.p1.m4" class="ltx_Math" alttext="\tt{treatment}" display="inline"><mi>𝚝𝚛𝚎𝚊𝚝𝚖𝚎𝚗𝚝</mi></math>.</p>
</div>
<div id="S4.SS2.p2" class="ltx_para">
<p class="ltx_p">Table <a href="#S5.T2" title="Table 2 ‣ Non-subjective power analysis to detect G×E interactions in Genome-Wide Association Studies in presence of confounding factor" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a> shows the results obtained observing the confounding factor. In this context, AUC estimated with PLINK SNP<math id="S4.SS2.p2.m1" class="ltx_Math" alttext="\times\tt{treatment}" display="inline"><mrow><mi></mi><mo lspace="0.222em" rspace="0.222em">×</mo><mi>𝚝𝚛𝚎𝚊𝚝𝚖𝚎𝚗𝚝</mi></mrow></math> are equal to 1, as expected given the strength of the simulated G<math id="S4.SS2.p2.m2" class="ltx_Math" alttext="\times" display="inline"><mo>×</mo></math>E interaction. This results show that the approach accounting for the interaction is better than approach accounting only for the SNP and demonstrate the importance of accounting for G<math id="S4.SS2.p2.m3" class="ltx_Math" alttext="\times" display="inline"><mo>×</mo></math>E interaction in a process of consideration of a confounding factor.</p>
</div>
<div id="S4.SS2.p3" class="ltx_para">
<p class="ltx_p">In contrast, the AUC estimated with PLINK SNP are almost equal to the AUC estimated when the confounding factor <math id="S4.SS2.p3.m1" class="ltx_Math" alttext="\tt{treatment}" display="inline"><mi>𝚝𝚛𝚎𝚊𝚝𝚖𝚎𝚗𝚝</mi></math> is latent and instead the covariate <math id="S4.SS2.p3.m2" class="ltx_Math" alttext="\tt{bmi}" display="inline"><mi>𝚋𝚖𝚒</mi></math> is observed. As expected, Fast-LMM gives similar results to PLINK SNP.
For the RF, we can only observe a slight improvement (<math id="S4.SS2.p3.m3" class="ltx_Math" alttext="1" display="inline"><mn>1</mn></math> to <math id="S4.SS2.p3.m4" class="ltx_Math" alttext="3\%" display="inline"><mrow><mn>3</mn><mo>%</mo></mrow></math> of AUC). This is due to the fact that, like for Fast-LMM or PLINK SNP, the RF approach does not consider explicitly interactions with the covariates. However, RF are known to be able to capture complex non-linear interaction between covariates which tend to be co-selected in the same trees. In our example, this alleged feature clearly shows its limits.</p>
</div>
</section>
<section id="S4.SS3" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">4.3 </span>Further considerations</h3>

<div id="S4.SS3.p1" class="ltx_para">
<p class="ltx_p">An interesting property of Fast-LMM is it accounts for population structure by introducing random effects as predictors. In this context, analysis were performed with Fast-LMM, with no principal components as covariates. We verified (results not shown) that accounting explicitly for principal components gives similar results. In the same way, we performed analysis with PLINK SNP<math id="S4.SS3.p1.m1" class="ltx_Math" alttext="\times\tt{bmi}" display="inline"><mrow><mi></mi><mo lspace="0.222em" rspace="0.222em">×</mo><mi>𝚋𝚖𝚒</mi></mrow></math> considering explicitly the population of belonging instead of principal components and found similar results than considering principal components (results not shown).</p>
</div>
<div id="S4.SS3.p2" class="ltx_para">
<p class="ltx_p">Another question that may arise is about the signal localization. Indeed, presence of confounding factor makes very difficult to identify the causal SNP. Figure <a href="#S5.F2" title="Figure 2 ‣ Non-subjective power analysis to detect G×E interactions in Genome-Wide Association Studies in presence of confounding factor" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a> shows the Manhattan plots obtained performing PLINK SNP<math id="S4.SS3.p2.m1" class="ltx_Math" alttext="\times" display="inline"><mo>×</mo></math>COV analysis on Chromosome 6 from one simulation under H1. The vertical red line indicates the causal SNP location. On figure <a href="#S5.F2.sf1" title="In Figure 2 ‣ Non-subjective power analysis to detect G×E interactions in Genome-Wide Association Studies in presence of confounding factor" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2(a)</span></a> analysis was performed observing the confounding factor <math id="S4.SS3.p2.m2" class="ltx_Math" alttext="\tt{treatment}" display="inline"><mi>𝚝𝚛𝚎𝚊𝚝𝚖𝚎𝚗𝚝</mi></math> (i.e. using PLINK SNP<math id="S4.SS3.p2.m3" class="ltx_Math" alttext="\times\tt{treatment}" display="inline"><mrow><mi></mi><mo lspace="0.222em" rspace="0.222em">×</mo><mi>𝚝𝚛𝚎𝚊𝚝𝚖𝚎𝚗𝚝</mi></mrow></math> approach) and on figure  <a href="#S5.F2.sf2" title="In Figure 2 ‣ Non-subjective power analysis to detect G×E interactions in Genome-Wide Association Studies in presence of confounding factor" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2(b)</span></a> analysis was performed in presence of confounding factor. (i.e. using PLINK SNP<math id="S4.SS3.p2.m4" class="ltx_Math" alttext="\times\tt{bmi}" display="inline"><mrow><mi></mi><mo lspace="0.222em" rspace="0.222em">×</mo><mi>𝚋𝚖𝚒</mi></mrow></math> approach). When the confounding factor is known, the location detected by the signal is the same as the causal SNP location. In contrast, in presence of confounding factor, there is no signal clearly detected.</p>
</div>
<div class="ltx_pagination ltx_role_newpage"></div>
</section>
</section>
<section id="S5" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">5 </span>Conclusion</h2>

<div id="S5.p1" class="ltx_para">
<p class="ltx_p">This article presents a simulated dataset - based on the HapMap project data with real genotypes and population structures - which can be helpful in studying the impact of a confounding factor on the detection of G<math id="S5.p1.m1" class="ltx_Math" alttext="\times" display="inline"><mo>×</mo></math>E interactions in GWAS. Furthermore, the proposed dataset was used to conduct a non-subjective power analysis using three popular GWAS statistical methods (PLINK, Random Forests and Fast-LMM) and four corresponding summary statistics. The non-subjectivity comes from the fact that the phenotypes under H1 as well as under H0 were simulated with <math id="S5.p1.m2" class="ltx_Math" alttext="{\tt waffect}" display="inline"><mi>𝚠𝚊𝚏𝚏𝚎𝚌𝚝</mi></math> which does not require any additional information other than the original genotypes. The dataset will be made available on ENA at the time of publication, so that all replication will be feasible.</p>
</div>
<div id="S5.p2" class="ltx_para">
<p class="ltx_p">The disease model was built with a strong interaction between the causal SNP (which was chosen arbitrarily) and an unobserved environmental exposure to a treatment and no main effect given by the causal SNP or the latent exposure. Obviously, the high relative risk resulting from the considered G<math id="S5.p2.m1" class="ltx_Math" alttext="\times" display="inline"><mo>×</mo></math>E interaction and the absence of a marginal SNP effect are unrealistic. However, the confounding factor causes such a loss of detection power that it is necessary to study such typical case in first place. Indeed, analysis performed on the whole chromosome 6 shows a poor signal detection power for the four approaches considered in presence of confounding factor. Furthermore, we show that when analysis is performed regardless the interaction, the fact to observe or not the confounding factor <math id="S5.p2.m2" class="ltx_Math" alttext="\tt{treatment}" display="inline"><mi>𝚝𝚛𝚎𝚊𝚝𝚖𝚎𝚗𝚝</mi></math> has no impact on the detection power. These results highlight the importance of taking into account the G<math id="S5.p2.m3" class="ltx_Math" alttext="\times" display="inline"><mo>×</mo></math>E interactions at the risk of finding no signal at all.</p>
</div>
<div id="S5.p3" class="ltx_para">
<p class="ltx_p">In this work, we chose to focus on three popular methods belonging to different families of statistical techniques.
Unfortunately, the current implementation of Fast-LMM does not allow to account for G<math id="S5.p3.m1" class="ltx_Math" alttext="\times" display="inline"><mo>×</mo></math>E interactions. The proposed dataset has the potential to provide a good framework to develop further features of Fast-LMM enabling it to account for such interactions. As an alternative, we considered the idea to precompute a full covariate matrix including interaction to Fast-LMM, but this approach was finally discarded since there was no way to force fast-LMM to perform the appropriate significance tests. Similar consideration hold for the RF which are not specifically designed to deal with explicit interactions.</p>
</div>
<div id="S5.p4" class="ltx_para">
<p class="ltx_p">Other methods based on the linear model are known for case control data, in order to detect G<math id="S5.p4.m1" class="ltx_Math" alttext="\times" display="inline"><mo>×</mo></math>E interactions. Kraft and al. <cite class="ltx_cite ltx_citemacro_citep">(Kraft et al., <a href="#bib.bib13" title="" class="ltx_ref">2007</a>)</cite> proposed a powerful 2-df joint test of marginal association and G<math id="S5.p4.m2" class="ltx_Math" alttext="\times" display="inline"><mo>×</mo></math>E interaction. Shortly after, Mucray and al. <cite class="ltx_cite ltx_citemacro_citep">(Murcray et al., <a href="#bib.bib17" title="" class="ltx_ref">2009</a>)</cite> proposed a 2-step approach for detecting G<math id="S5.p4.m3" class="ltx_Math" alttext="\times" display="inline"><mo>×</mo></math>E interaction in GWA studies.
Dai and al.<cite class="ltx_cite ltx_citemacro_citep">(Dai et al., <a href="#bib.bib6" title="" class="ltx_ref">2012</a>)</cite> proposed a new way to combine the test of marginal genetic effect and the test of G<math id="S5.p4.m4" class="ltx_Math" alttext="\times" display="inline"><mo>×</mo></math>E interaction, by exploiting the independence between the two tests.
While these methods have demonstrated their efficiency, their performance was assessed through simulations that do not account for realistic complexity such as the inclusion of confounding factor and/or of complicated dependance structures between individuals as well as between loci.
Testing the detection power of these tests on our realistic simulated dataset could then be an interesting development. Results obtained in this paper already seem to show that methods based on the linear model have poor power. As a consequence, we can expect that the tests mentioned above will not perform well either.</p>
</div>
<div id="S5.p5" class="ltx_para">
<p class="ltx_p">In conclusion, efforts should be put in developing methods able to take into account confounding factors such as a potential latent exposure.</p>
</div>
<div class="ltx_pagination ltx_role_newpage"></div>
</section>
<section id="bib" class="ltx_bibliography">
<h2 class="ltx_title ltx_title_bibliography">References</h2>

<ul class="ltx_biblist">
      
<li id="bib.bib1" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Astle and Balding (2009)</span>
<span class="ltx_bibblock">
William Astle and David J Balding.

</span>
<span class="ltx_bibblock">Population structure and cryptic relatedness in genetic association
studies.

</span>
<span class="ltx_bibblock"><em class="ltx_emph ltx_font_italic">Statistical Science</em>, 24(4):451–471, 2009.

</span>
</li>
      
<li id="bib.bib2" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Boulesteix et al. (2012)</span>
<span class="ltx_bibblock">
Anne-Laure Boulesteix, Silke Janitza, Jochen Kruppa, and Inke R König.

</span>
<span class="ltx_bibblock">Overview of random forest methodology and practical guidance with
emphasis on computational biology and bioinformatics.

</span>
<span class="ltx_bibblock"><em class="ltx_emph ltx_font_italic">Wiley Interdisciplinary Reviews: Data Mining and Knowledge
Discovery</em>, 2(6):493–507, 2012.

</span>
</li>
      
<li id="bib.bib3" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Breiman (2001)</span>
<span class="ltx_bibblock">
Leo Breiman.

</span>
<span class="ltx_bibblock">Random forests.

</span>
<span class="ltx_bibblock"><em class="ltx_emph ltx_font_italic">Machine learning</em>, 45(1):5–32, 2001.

</span>
</li>
      
<li id="bib.bib4" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Chiolero et al. (2008)</span>
<span class="ltx_bibblock">
Arnaud Chiolero, David Faeh, Fred Paccaud, and Jacques Cornuz.

</span>
<span class="ltx_bibblock">Consequences of smoking for body weight, body fat distribution, and
insulin resistance.

</span>
<span class="ltx_bibblock"><em class="ltx_emph ltx_font_italic">The American journal of clinical nutrition</em>, 87(4):801–809, 2008.

</span>
</li>
      
<li id="bib.bib5" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Christopoulou and Lillard (2011)</span>
<span class="ltx_bibblock">
Rebekka Christopoulou and Dean R Lillard.

</span>
<span class="ltx_bibblock">The role of culture in smoking behavior: evidence from british
immigrants in australia, south africa, and the us.

</span>
<span class="ltx_bibblock">Technical report, Cornell University, 2011.

</span>
</li>
      
<li id="bib.bib6" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Dai et al. (2012)</span>
<span class="ltx_bibblock">
James Y Dai, Benjamin A Logsdon, Ying Huang, Li Hsu, Alexander P Reiner, Ross L
Prentice, and Charles Kooperberg.

</span>
<span class="ltx_bibblock">Simultaneously testing for marginal genetic association and
gene-environment interaction.

</span>
<span class="ltx_bibblock"><em class="ltx_emph ltx_font_italic">American journal of epidemiology</em>, 176(2):164–173, 2012.

</span>
</li>
      
<li id="bib.bib7" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Deng and Runger (2012)</span>
<span class="ltx_bibblock">
Houtao Deng and George Runger.

</span>
<span class="ltx_bibblock">Feature selection via regularized trees.

</span>
<span class="ltx_bibblock"><em class="ltx_emph ltx_font_italic">The 2012 International Joint Conference on Neural Networks
(IJCNN)</em>, 2012.

</span>
</li>
      
<li id="bib.bib8" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Garcia-Closas et al. (2010)</span>
<span class="ltx_bibblock">
M Garcia-Closas, K Jacobs, P Kraft, and N Chatterjee.

</span>
<span class="ltx_bibblock">Analysis of epidemiologic studies of genetic effects and
gene-environment interactions.

</span>
<span class="ltx_bibblock"><em class="ltx_emph ltx_font_italic">IARC scientific publications</em>, 163(163):281–301, 2010.

</span>
</li>
      
<li id="bib.bib9" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Gibbs et al. (2003)</span>
<span class="ltx_bibblock">
Richard A Gibbs, John W Belmont, Paul Hardenbol, Thomas D Willis, Fuli Yu,
Huanming Yang, Lan-Yang Ch’ang, Wei Huang, Bin Liu, Yan Shen, et al.

</span>
<span class="ltx_bibblock">The international hapmap project.

</span>
<span class="ltx_bibblock"><em class="ltx_emph ltx_font_italic">Nature</em>, 426(6968):789–796, 2003.

</span>
</li>
      
<li id="bib.bib10" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Goldstein et al. (2010)</span>
<span class="ltx_bibblock">
Benjamin A Goldstein, Alan E Hubbard, Adele Cutler, and Lisa F Barcellos.

</span>
<span class="ltx_bibblock">An application of Random Forests to a genome-wide association
dataset: Methodological considerations &amp; new findings.

</span>
<span class="ltx_bibblock"><em class="ltx_emph ltx_font_italic">BMC genetics</em>, 11(1):49, 2010.

</span>
</li>
      
<li id="bib.bib11" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Hamza et al. (2011)</span>
<span class="ltx_bibblock">
Taye H Hamza, Honglei Chen, Erin M Hill-Burns, Shannon L Rhodes, Jennifer
Montimurro, Denise M Kay, Albert Tenesa, Victoria I Kusel, Patricia Sheehan,
Muthukrishnan Eaaswarkhanth, et al.

</span>
<span class="ltx_bibblock">Genome-wide gene-environment study identifies glutamate receptor gene
grin2a as a parkinson’s disease modifier gene via interaction with coffee.

</span>
<span class="ltx_bibblock"><em class="ltx_emph ltx_font_italic">PLoS genetics</em>, 7(8):e1002237, 2011.

</span>
</li>
      
<li id="bib.bib12" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Hoffman (2013)</span>
<span class="ltx_bibblock">
Gabriel E. Hoffman.

</span>
<span class="ltx_bibblock">Correcting for population structure and kinship using the linear
mixed model: Theory and extensions.

</span>
<span class="ltx_bibblock"><em class="ltx_emph ltx_font_italic">PLoS ONE</em>, 8(10):e75707, 10 2013.

</span>
<span class="ltx_bibblock">doi: <span class="ltx_ref ltx_nolink ltx_Url ltx_ref_self">10.1371/journal.pone.0075707</span>.

</span>
<span class="ltx_bibblock">URL <a href="http://dx.doi.org/10.1371%2Fjournal.pone.0075707" title="" class="ltx_ref ltx_url ltx_font_typewriter">http://dx.doi.org/10.1371%2Fjournal.pone.0075707</a>.

</span>
</li>
      
<li id="bib.bib13" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Kraft et al. (2007)</span>
<span class="ltx_bibblock">
Peter Kraft, Y-C Yen, Daniel O Stram, John Morrison, and W James Gauderman.

</span>
<span class="ltx_bibblock">Exploiting gene-environment interaction to detect genetic
associations.

</span>
<span class="ltx_bibblock"><em class="ltx_emph ltx_font_italic">Human heredity</em>, 63(2):111–119, 2007.

</span>
</li>
      
<li id="bib.bib14" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Liaw and Wiener (2002)</span>
<span class="ltx_bibblock">
Andy Liaw and Matthew Wiener.

</span>
<span class="ltx_bibblock">Classification and regression by randomforest.

</span>
<span class="ltx_bibblock"><em class="ltx_emph ltx_font_italic">R News</em>, 2(3):18–22, 2002.

</span>
<span class="ltx_bibblock">URL <a href="http://CRAN.R-project.org/doc/Rnews/" title="" class="ltx_ref ltx_url ltx_font_typewriter">http://CRAN.R-project.org/doc/Rnews/</a>.

</span>
</li>
      
<li id="bib.bib15" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Lippert et al. (2011)</span>
<span class="ltx_bibblock">
Christoph Lippert, Jennifer Listgarten, Ying Liu, Carl M Kadie, Robert I
Davidson, and David Heckerman.

</span>
<span class="ltx_bibblock">Fast linear mixed models for genome-wide association studies.

</span>
<span class="ltx_bibblock"><em class="ltx_emph ltx_font_italic">Nature Methods</em>, 8(10):833–835, 2011.

</span>
</li>
      
<li id="bib.bib16" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Metz (1978)</span>
<span class="ltx_bibblock">
Charles E Metz.

</span>
<span class="ltx_bibblock">Basic principles of roc analysis.

</span>
<span class="ltx_bibblock">In <em class="ltx_emph ltx_font_italic">Seminars in nuclear medicine</em>, volume 8, pages 283–298.
Elsevier, 1978.

</span>
</li>
      
<li id="bib.bib17" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Murcray et al. (2009)</span>
<span class="ltx_bibblock">
Cassandra E Murcray, Juan Pablo Lewinger, and W James Gauderman.

</span>
<span class="ltx_bibblock">Gene-environment interaction in genome-wide association studies.

</span>
<span class="ltx_bibblock"><em class="ltx_emph ltx_font_italic">American journal of epidemiology</em>, 169(2):219–226, 2009.

</span>
</li>
      
<li id="bib.bib18" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Perduca et al. (2012)</span>
<span class="ltx_bibblock">
Vittorio Perduca, Christine Sinoquet, Raphaël Mourad, and Gregory Nuel.

</span>
<span class="ltx_bibblock">Alternative Methods for H1 Simulations in Genome-Wide
Association Studies.

</span>
<span class="ltx_bibblock"><em class="ltx_emph ltx_font_italic">Human Heredity</em>, 73(2):95–104, 2012.

</span>
</li>
      
<li id="bib.bib19" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Purcell et al. (2007)</span>
<span class="ltx_bibblock">
Shaun Purcell, Benjamin Neale, Kathe Todd-Brown, Lori Thomas, Manuel AR
Ferreira, David Bender, Julian Maller, Pamela Sklar, Paul IW De Bakker,
Mark J Daly, et al.

</span>
<span class="ltx_bibblock">PLINK: a tool set for whole-genome association and population-based
linkage analyses.

</span>
<span class="ltx_bibblock"><em class="ltx_emph ltx_font_italic">The American Journal of Human Genetics</em>, 81(3):559–575, 2007.

</span>
</li>
      
<li id="bib.bib20" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">R Core Team (2013)</span>
<span class="ltx_bibblock">
R Core Team.

</span>
<span class="ltx_bibblock"><em class="ltx_emph ltx_font_italic">R: A Language and Environment for Statistical Computing</em>.

</span>
<span class="ltx_bibblock">R Foundation for Statistical Computing, Vienna, Austria, 2013.

</span>
<span class="ltx_bibblock">URL <a href="http://www.R-project.org" title="" class="ltx_ref ltx_url ltx_font_typewriter">http://www.R-project.org</a>.

</span>
</li>
      
<li id="bib.bib21" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Robin et al. (2011)</span>
<span class="ltx_bibblock">
Xavier Robin, Natacha Turck, Alexandre Hainard, Natalia Tiberti,
Frédérique Lisacek, Jean-Charles Sanchez, and Markus Müller.

</span>
<span class="ltx_bibblock">proc: an open-source package for r and s+ to analyze and compare roc
curves.

</span>
<span class="ltx_bibblock"><em class="ltx_emph ltx_font_italic">BMC bioinformatics</em>, 12(1):77, 2011.

</span>
</li>
      
<li id="bib.bib22" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Rothman et al. (2010)</span>
<span class="ltx_bibblock">
Nathaniel Rothman, Montserrat Garcia-Closas, Nilanjan Chatterjee, Nuria Malats,
Xifeng Wu, Jonine D Figueroa, Francisco X Real, David Van Den Berg, Giuseppe
Matullo, Dalsu Baris, et al.

</span>
<span class="ltx_bibblock">A multi-stage genome-wide association study of bladder cancer
identifies multiple susceptibility loci.

</span>
<span class="ltx_bibblock"><em class="ltx_emph ltx_font_italic">Nature genetics</em>, 42(11):978–984, 2010.

</span>
</li>
      
<li id="bib.bib23" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Spencer et al. (2009)</span>
<span class="ltx_bibblock">
Chris CA Spencer, Zhan Su, Peter Donnelly, and Jonathan Marchini.

</span>
<span class="ltx_bibblock">Designing genome-wide association studies: sample size, power,
imputation, and the choice of genotyping chip.

</span>
<span class="ltx_bibblock"><em class="ltx_emph ltx_font_italic">PLoS genetics</em>, 5(5):e1000477, 2009.

</span>
</li>
      
<li id="bib.bib24" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Su et al. (2011)</span>
<span class="ltx_bibblock">
Zhan Su, Jonathan Marchini, and Peter Donnelly.

</span>
<span class="ltx_bibblock">HAPGEN2: simulation of multiple disease SNPs.

</span>
<span class="ltx_bibblock"><em class="ltx_emph ltx_font_italic">Bioinformatics</em>, 27(16):2304–2305, 2011.

</span>
</li>
      
<li id="bib.bib25" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Thorisson et al. (2005)</span>
<span class="ltx_bibblock">
Gudmundur A Thorisson, Albert V Smith, Lalitha Krishnan, and Lincoln D Stein.

</span>
<span class="ltx_bibblock">The international hapmap project web site.

</span>
<span class="ltx_bibblock"><em class="ltx_emph ltx_font_italic">Genome research</em>, 15(11):1592–1593, 2005.

</span>
</li>
      
<li id="bib.bib26" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Tsai et al. (2008)</span>
<span class="ltx_bibblock">
Yi-Wen Tsai, Tzu-I Tsai, Chung-Lin Yang, and Ken N Kuo.

</span>
<span class="ltx_bibblock">Gender differences in smoking behaviors in an asian population.

</span>
<span class="ltx_bibblock"><em class="ltx_emph ltx_font_italic">Journal of Women’s Health</em>, 17(6):971–978,
2008.

</span>
</li>
      
<li id="bib.bib27" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">VanderWeele et al. (2012)</span>
<span class="ltx_bibblock">
Tyler J VanderWeele, Bhramar Mukherjee, and Jinbo Chen.

</span>
<span class="ltx_bibblock">Sensitivity analysis for interactions under unmeasured confounding.

</span>
<span class="ltx_bibblock"><em class="ltx_emph ltx_font_italic">Statistics in medicine</em>, 31(22):2552–2564,
2012.

</span>
</li>
      
<li id="bib.bib28" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">VanderWeele et al. (2013)</span>
<span class="ltx_bibblock">
Tyler J VanderWeele, Yi-An Ko, and Bhramar Mukherjee.

</span>
<span class="ltx_bibblock">Environmental confounding in gene-environment interaction studies.

</span>
<span class="ltx_bibblock"><em class="ltx_emph ltx_font_italic">American Journal of Epidemiology</em>, 2013.

</span>
</li>
    
</ul>
</section>
<div class="ltx_pagination ltx_role_newpage"></div>
<figure id="S5.T1" class="ltx_table">
<table class="ltx_tabular ltx_centering ltx_guessed_headers ltx_align_middle">
<thead class="ltx_thead">
<tr class="ltx_tr">
<th class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_th_row ltx_border_t"><span class="ltx_text" style="font-size:90%;">AUC (%)</span></th>
<th class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_t"><span class="ltx_text" style="font-size:90%;">PLINK SNP</span></th>
<th class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_t">
<span class="ltx_text" style="font-size:90%;">PLINK SNP</span><math id="S5.T1.m1" class="ltx_Math" alttext="\times" display="inline"><mo mathsize="90%">×</mo></math><span class="ltx_text" style="font-size:90%;">bmi</span>
</th>
<th class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_t"><span class="ltx_text" style="font-size:90%;">RF</span></th>
<th class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_t"><span class="ltx_text" style="font-size:90%;">Fast-LMM</span></th>
</tr>
</thead>
<tbody class="ltx_tbody">
<tr class="ltx_tr">
<th class="ltx_td ltx_align_center ltx_th ltx_th_row ltx_border_t"><span class="ltx_text" style="font-size:90%;">whole chromosome 6</span></th>
<td class="ltx_td ltx_align_center ltx_border_t"><math id="S5.T1.m2" class="ltx_Math" alttext="64.69\ [59.26-70.13]" display="inline"><mrow><mn mathsize="90%">64.69</mn><mo lspace="0.450em">⁢</mo><mrow><mo maxsize="90%" minsize="90%">[</mo><mrow><mn mathsize="90%">59.26</mn><mo mathsize="90%">−</mo><mn mathsize="90%">70.13</mn></mrow><mo maxsize="90%" minsize="90%">]</mo></mrow></mrow></math></td>
<td class="ltx_td ltx_align_center ltx_border_t"><math id="S5.T1.m3" class="ltx_Math" alttext="56.39\ [50.69-62.08]" display="inline"><mrow><mn mathsize="90%">56.39</mn><mo lspace="0.450em">⁢</mo><mrow><mo maxsize="90%" minsize="90%">[</mo><mrow><mn mathsize="90%">50.69</mn><mo mathsize="90%">−</mo><mn mathsize="90%">62.08</mn></mrow><mo maxsize="90%" minsize="90%">]</mo></mrow></mrow></math></td>
<td class="ltx_td ltx_align_center ltx_border_t"><math id="S5.T1.m4" class="ltx_Math" alttext="66.23\ [60.93-71.55]" display="inline"><mrow><mn mathsize="90%">66.23</mn><mo lspace="0.450em">⁢</mo><mrow><mo maxsize="90%" minsize="90%">[</mo><mrow><mn mathsize="90%">60.93</mn><mo mathsize="90%">−</mo><mn mathsize="90%">71.55</mn></mrow><mo maxsize="90%" minsize="90%">]</mo></mrow></mrow></math></td>
<td class="ltx_td ltx_align_center ltx_border_t"><math id="S5.T1.m5" class="ltx_Math" alttext="61.91\ [56.42-67.41]" display="inline"><mrow><mn mathsize="90%">61.91</mn><mo lspace="0.450em">⁢</mo><mrow><mo maxsize="90%" minsize="90%">[</mo><mrow><mn mathsize="90%">56.42</mn><mo mathsize="90%">−</mo><mn mathsize="90%">67.41</mn></mrow><mo maxsize="90%" minsize="90%">]</mo></mrow></mrow></math></td>
</tr>
<tr class="ltx_tr">
<th class="ltx_td ltx_align_center ltx_th ltx_th_row"><span class="ltx_text" style="font-size:90%;">8 000 SNPs region</span></th>
<td class="ltx_td ltx_align_center"><math id="S5.T1.m6" class="ltx_Math" alttext="72.04\ [67.03-77.05]" display="inline"><mrow><mn mathsize="90%">72.04</mn><mo lspace="0.450em">⁢</mo><mrow><mo maxsize="90%" minsize="90%">[</mo><mrow><mn mathsize="90%">67.03</mn><mo mathsize="90%">−</mo><mn mathsize="90%">77.05</mn></mrow><mo maxsize="90%" minsize="90%">]</mo></mrow></mrow></math></td>
<td class="ltx_td ltx_align_center"><math id="S5.T1.m7" class="ltx_Math" alttext="55.32\ [49.60-61.04]" display="inline"><mrow><mn mathsize="90%">55.32</mn><mo lspace="0.450em">⁢</mo><mrow><mo maxsize="90%" minsize="90%">[</mo><mrow><mn mathsize="90%">49.60</mn><mo mathsize="90%">−</mo><mn mathsize="90%">61.04</mn></mrow><mo maxsize="90%" minsize="90%">]</mo></mrow></mrow></math></td>
<td class="ltx_td ltx_align_center"><math id="S5.T1.m8" class="ltx_Math" alttext="71.99\ [67.07-76.91]" display="inline"><mrow><mn mathsize="90%">71.99</mn><mo lspace="0.450em">⁢</mo><mrow><mo maxsize="90%" minsize="90%">[</mo><mrow><mn mathsize="90%">67.07</mn><mo mathsize="90%">−</mo><mn mathsize="90%">76.91</mn></mrow><mo maxsize="90%" minsize="90%">]</mo></mrow></mrow></math></td>
<td class="ltx_td ltx_align_center"><math id="S5.T1.m9" class="ltx_Math" alttext="68.96\ [63.84-74.07]" display="inline"><mrow><mn mathsize="90%">68.96</mn><mo lspace="0.450em">⁢</mo><mrow><mo maxsize="90%" minsize="90%">[</mo><mrow><mn mathsize="90%">63.84</mn><mo mathsize="90%">−</mo><mn mathsize="90%">74.07</mn></mrow><mo maxsize="90%" minsize="90%">]</mo></mrow></mrow></math></td>
</tr>
<tr class="ltx_tr">
<th class="ltx_td ltx_align_center ltx_th ltx_th_row"><span class="ltx_text" style="font-size:90%;">2 000 SNPs region</span></th>
<td class="ltx_td ltx_align_center"><math id="S5.T1.m10" class="ltx_Math" alttext="74.44\ [69.52-79.35]" display="inline"><mrow><mn mathsize="90%">74.44</mn><mo lspace="0.450em">⁢</mo><mrow><mo maxsize="90%" minsize="90%">[</mo><mrow><mn mathsize="90%">69.52</mn><mo mathsize="90%">−</mo><mn mathsize="90%">79.35</mn></mrow><mo maxsize="90%" minsize="90%">]</mo></mrow></mrow></math></td>
<td class="ltx_td ltx_align_center"><math id="S5.T1.m11" class="ltx_Math" alttext="58.05\ [52.41-63.70]" display="inline"><mrow><mn mathsize="90%">58.05</mn><mo lspace="0.450em">⁢</mo><mrow><mo maxsize="90%" minsize="90%">[</mo><mrow><mn mathsize="90%">52.41</mn><mo mathsize="90%">−</mo><mn mathsize="90%">63.70</mn></mrow><mo maxsize="90%" minsize="90%">]</mo></mrow></mrow></math></td>
<td class="ltx_td ltx_align_center"><math id="S5.T1.m12" class="ltx_Math" alttext="76.15\ [71.49-80.82]" display="inline"><mrow><mn mathsize="90%">76.15</mn><mo lspace="0.450em">⁢</mo><mrow><mo maxsize="90%" minsize="90%">[</mo><mrow><mn mathsize="90%">71.49</mn><mo mathsize="90%">−</mo><mn mathsize="90%">80.82</mn></mrow><mo maxsize="90%" minsize="90%">]</mo></mrow></mrow></math></td>
<td class="ltx_td ltx_align_center"><math id="S5.T1.m13" class="ltx_Math" alttext="71.36\ [66.35-76.36]" display="inline"><mrow><mn mathsize="90%">71.36</mn><mo lspace="0.450em">⁢</mo><mrow><mo maxsize="90%" minsize="90%">[</mo><mrow><mn mathsize="90%">66.35</mn><mo mathsize="90%">−</mo><mn mathsize="90%">76.36</mn></mrow><mo maxsize="90%" minsize="90%">]</mo></mrow></mrow></math></td>
</tr>
<tr class="ltx_tr">
<th class="ltx_td ltx_align_center ltx_th ltx_th_row"><span class="ltx_text" style="font-size:90%;">800 SNPs regions</span></th>
<td class="ltx_td ltx_align_center"><math id="S5.T1.m14" class="ltx_Math" alttext="81.78\ [77.54-86.02]" display="inline"><mrow><mn mathsize="90%">81.78</mn><mo lspace="0.450em">⁢</mo><mrow><mo maxsize="90%" minsize="90%">[</mo><mrow><mn mathsize="90%">77.54</mn><mo mathsize="90%">−</mo><mn mathsize="90%">86.02</mn></mrow><mo maxsize="90%" minsize="90%">]</mo></mrow></mrow></math></td>
<td class="ltx_td ltx_align_center"><math id="S5.T1.m15" class="ltx_Math" alttext="60.24\ [54.66-65.81]" display="inline"><mrow><mn mathsize="90%">60.24</mn><mo lspace="0.450em">⁢</mo><mrow><mo maxsize="90%" minsize="90%">[</mo><mrow><mn mathsize="90%">54.66</mn><mo mathsize="90%">−</mo><mn mathsize="90%">65.81</mn></mrow><mo maxsize="90%" minsize="90%">]</mo></mrow></mrow></math></td>
<td class="ltx_td ltx_align_center"><math id="S5.T1.m16" class="ltx_Math" alttext="79.48\ [75.10-83.87]" display="inline"><mrow><mn mathsize="90%">79.48</mn><mo lspace="0.450em">⁢</mo><mrow><mo maxsize="90%" minsize="90%">[</mo><mrow><mn mathsize="90%">75.10</mn><mo mathsize="90%">−</mo><mn mathsize="90%">83.87</mn></mrow><mo maxsize="90%" minsize="90%">]</mo></mrow></mrow></math></td>
<td class="ltx_td ltx_align_center"><math id="S5.T1.m17" class="ltx_Math" alttext="80.65\ [76.48-84.82]" display="inline"><mrow><mn mathsize="90%">80.65</mn><mo lspace="0.450em">⁢</mo><mrow><mo maxsize="90%" minsize="90%">[</mo><mrow><mn mathsize="90%">76.48</mn><mo mathsize="90%">−</mo><mn mathsize="90%">84.82</mn></mrow><mo maxsize="90%" minsize="90%">]</mo></mrow></mrow></math></td>
</tr>
<tr class="ltx_tr">
<th class="ltx_td ltx_align_center ltx_th ltx_th_row"><span class="ltx_text" style="font-size:90%;">200 SNPs region</span></th>
<td class="ltx_td ltx_align_center"><math id="S5.T1.m18" class="ltx_Math" alttext="88.50\ [85.27-91.72]" display="inline"><mrow><mn mathsize="90%">88.50</mn><mo lspace="0.450em">⁢</mo><mrow><mo maxsize="90%" minsize="90%">[</mo><mrow><mn mathsize="90%">85.27</mn><mo mathsize="90%">−</mo><mn mathsize="90%">91.72</mn></mrow><mo maxsize="90%" minsize="90%">]</mo></mrow></mrow></math></td>
<td class="ltx_td ltx_align_center"><math id="S5.T1.m19" class="ltx_Math" alttext="68.62\ [63.38-73.85]" display="inline"><mrow><mn mathsize="90%">68.62</mn><mo lspace="0.450em">⁢</mo><mrow><mo maxsize="90%" minsize="90%">[</mo><mrow><mn mathsize="90%">63.38</mn><mo mathsize="90%">−</mo><mn mathsize="90%">73.85</mn></mrow><mo maxsize="90%" minsize="90%">]</mo></mrow></mrow></math></td>
<td class="ltx_td ltx_align_center"><math id="S5.T1.m20" class="ltx_Math" alttext="84.71\ [80.77-88.66]" display="inline"><mrow><mn mathsize="90%">84.71</mn><mo lspace="0.450em">⁢</mo><mrow><mo maxsize="90%" minsize="90%">[</mo><mrow><mn mathsize="90%">80.77</mn><mo mathsize="90%">−</mo><mn mathsize="90%">88.66</mn></mrow><mo maxsize="90%" minsize="90%">]</mo></mrow></mrow></math></td>
<td class="ltx_td ltx_align_center"><math id="S5.T1.m21" class="ltx_Math" alttext="86.72\ [83.28-90.17]" display="inline"><mrow><mn mathsize="90%">86.72</mn><mo lspace="0.450em">⁢</mo><mrow><mo maxsize="90%" minsize="90%">[</mo><mrow><mn mathsize="90%">83.28</mn><mo mathsize="90%">−</mo><mn mathsize="90%">90.17</mn></mrow><mo maxsize="90%" minsize="90%">]</mo></mrow></mrow></math></td>
</tr>
<tr class="ltx_tr">
<th class="ltx_td ltx_align_center ltx_th ltx_th_row ltx_border_b"><span class="ltx_text" style="font-size:90%;">causal SNP</span></th>
<td class="ltx_td ltx_align_center ltx_border_b"><math id="S5.T1.m22" class="ltx_Math" alttext="99.15\ [98.57-99.73]" display="inline"><mrow><mn mathsize="90%">99.15</mn><mo lspace="0.450em">⁢</mo><mrow><mo maxsize="90%" minsize="90%">[</mo><mrow><mn mathsize="90%">98.57</mn><mo mathsize="90%">−</mo><mn mathsize="90%">99.73</mn></mrow><mo maxsize="90%" minsize="90%">]</mo></mrow></mrow></math></td>
<td class="ltx_td ltx_align_center ltx_border_b"><math id="S5.T1.m23" class="ltx_Math" alttext="88.67\ [85.48-91.86]" display="inline"><mrow><mn mathsize="90%">88.67</mn><mo lspace="0.450em">⁢</mo><mrow><mo maxsize="90%" minsize="90%">[</mo><mrow><mn mathsize="90%">85.48</mn><mo mathsize="90%">−</mo><mn mathsize="90%">91.86</mn></mrow><mo maxsize="90%" minsize="90%">]</mo></mrow></mrow></math></td>
<td class="ltx_td ltx_align_center ltx_border_b"><math id="S5.T1.m24" class="ltx_Math" alttext="89.03\ [85.75-92.32]" display="inline"><mrow><mn mathsize="90%">89.03</mn><mo lspace="0.450em">⁢</mo><mrow><mo maxsize="90%" minsize="90%">[</mo><mrow><mn mathsize="90%">85.75</mn><mo mathsize="90%">−</mo><mn mathsize="90%">92.32</mn></mrow><mo maxsize="90%" minsize="90%">]</mo></mrow></mrow></math></td>
<td class="ltx_td ltx_align_center ltx_border_b"><math id="S5.T1.m25" class="ltx_Math" alttext="99.09\ [98.49-99.70]" display="inline"><mrow><mn mathsize="90%">99.09</mn><mo lspace="0.450em">⁢</mo><mrow><mo maxsize="90%" minsize="90%">[</mo><mrow><mn mathsize="90%">98.49</mn><mo mathsize="90%">−</mo><mn mathsize="90%">99.70</mn></mrow><mo maxsize="90%" minsize="90%">]</mo></mrow></mrow></math></td>
</tr>
</tbody>
</table>
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_table">Table 1: </span> Association analysis performed on Chromosome 6 with the four approaches, in presence of a confounding factor (i.e. covariate <math id="S5.T1.m28" class="ltx_Math" alttext="\tt{bmi}" display="inline"><mi>𝚋𝚖𝚒</mi></math> is observed but covariate <math id="S5.T1.m29" class="ltx_Math" alttext="\tt{treatment}" display="inline"><mi>𝚝𝚛𝚎𝚊𝚝𝚖𝚎𝚗𝚝</mi></math> is unknown) . Restricted regions are centered on causal SNP</figcaption>
</figure>
<div class="ltx_pagination ltx_role_newpage"></div>
<figure id="S5.T2" class="ltx_table">
<table class="ltx_tabular ltx_centering ltx_guessed_headers ltx_align_middle">
<thead class="ltx_thead">
<tr class="ltx_tr">
<th class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_th_row ltx_border_t"><span class="ltx_text" style="font-size:90%;">AUC (%)</span></th>
<th class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_t"><span class="ltx_text" style="font-size:90%;">PLINK SNP</span></th>
<th class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_t">
<span class="ltx_text" style="font-size:90%;">PLINK SNP</span><math id="S5.T2.m1" class="ltx_Math" alttext="\times" display="inline"><mo mathsize="90%">×</mo></math><span class="ltx_text" style="font-size:90%;">bmi</span>
</th>
<th class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_t"><span class="ltx_text" style="font-size:90%;">RF</span></th>
<th class="ltx_td ltx_align_center ltx_th ltx_th_column ltx_border_t"><span class="ltx_text" style="font-size:90%;">Fast-LMM</span></th>
</tr>
</thead>
<tbody class="ltx_tbody">
<tr class="ltx_tr">
<th class="ltx_td ltx_align_center ltx_th ltx_th_row ltx_border_t"><span class="ltx_text" style="font-size:90%;">whole chromosome 6</span></th>
<td class="ltx_td ltx_align_center ltx_border_t"><math id="S5.T2.m2" class="ltx_Math" alttext="63.35\ [57.84-68.85]" display="inline"><mrow><mn mathsize="90%">63.35</mn><mo lspace="0.450em">⁢</mo><mrow><mo maxsize="90%" minsize="90%">[</mo><mrow><mn mathsize="90%">57.84</mn><mo mathsize="90%">−</mo><mn mathsize="90%">68.85</mn></mrow><mo maxsize="90%" minsize="90%">]</mo></mrow></mrow></math></td>
<td class="ltx_td ltx_align_center ltx_border_t"><math id="S5.T2.m3" class="ltx_Math" alttext="99.97\ [99.93-100.0]" display="inline"><mrow><mn mathsize="90%">99.97</mn><mo lspace="0.450em">⁢</mo><mrow><mo maxsize="90%" minsize="90%">[</mo><mrow><mn mathsize="90%">99.93</mn><mo mathsize="90%">−</mo><mn mathsize="90%">100.0</mn></mrow><mo maxsize="90%" minsize="90%">]</mo></mrow></mrow></math></td>
<td class="ltx_td ltx_align_center ltx_border_t"><math id="S5.T2.m4" class="ltx_Math" alttext="67.66\ [62.44-72.88]" display="inline"><mrow><mn mathsize="90%">67.66</mn><mo lspace="0.450em">⁢</mo><mrow><mo maxsize="90%" minsize="90%">[</mo><mrow><mn mathsize="90%">62.44</mn><mo mathsize="90%">−</mo><mn mathsize="90%">72.88</mn></mrow><mo maxsize="90%" minsize="90%">]</mo></mrow></mrow></math></td>
<td class="ltx_td ltx_align_center ltx_border_t"><math id="S5.T2.m5" class="ltx_Math" alttext="61.54\ [56.02-67.06]" display="inline"><mrow><mn mathsize="90%">61.54</mn><mo lspace="0.450em">⁢</mo><mrow><mo maxsize="90%" minsize="90%">[</mo><mrow><mn mathsize="90%">56.02</mn><mo mathsize="90%">−</mo><mn mathsize="90%">67.06</mn></mrow><mo maxsize="90%" minsize="90%">]</mo></mrow></mrow></math></td>
</tr>
<tr class="ltx_tr">
<th class="ltx_td ltx_align_center ltx_th ltx_th_row"><span class="ltx_text" style="font-size:90%;">8 000 SNPs region</span></th>
<td class="ltx_td ltx_align_center"><math id="S5.T2.m6" class="ltx_Math" alttext="70.67\ [65.58-75.75]" display="inline"><mrow><mn mathsize="90%">70.67</mn><mo lspace="0.450em">⁢</mo><mrow><mo maxsize="90%" minsize="90%">[</mo><mrow><mn mathsize="90%">65.58</mn><mo mathsize="90%">−</mo><mn mathsize="90%">75.75</mn></mrow><mo maxsize="90%" minsize="90%">]</mo></mrow></mrow></math></td>
<td class="ltx_td ltx_align_center"><math id="S5.T2.m7" class="ltx_Math" alttext="100.0\ [99.99-100.0]" display="inline"><mrow><mn mathsize="90%">100.0</mn><mo lspace="0.450em">⁢</mo><mrow><mo maxsize="90%" minsize="90%">[</mo><mrow><mn mathsize="90%">99.99</mn><mo mathsize="90%">−</mo><mn mathsize="90%">100.0</mn></mrow><mo maxsize="90%" minsize="90%">]</mo></mrow></mrow></math></td>
<td class="ltx_td ltx_align_center"><math id="S5.T2.m8" class="ltx_Math" alttext="78.30\ [73.88-82.73]" display="inline"><mrow><mn mathsize="90%">78.30</mn><mo lspace="0.450em">⁢</mo><mrow><mo maxsize="90%" minsize="90%">[</mo><mrow><mn mathsize="90%">73.88</mn><mo mathsize="90%">−</mo><mn mathsize="90%">82.73</mn></mrow><mo maxsize="90%" minsize="90%">]</mo></mrow></mrow></math></td>
<td class="ltx_td ltx_align_center"><math id="S5.T2.m9" class="ltx_Math" alttext="68.02\ [62.82-73.23]" display="inline"><mrow><mn mathsize="90%">68.02</mn><mo lspace="0.450em">⁢</mo><mrow><mo maxsize="90%" minsize="90%">[</mo><mrow><mn mathsize="90%">62.82</mn><mo mathsize="90%">−</mo><mn mathsize="90%">73.23</mn></mrow><mo maxsize="90%" minsize="90%">]</mo></mrow></mrow></math></td>
</tr>
<tr class="ltx_tr">
<th class="ltx_td ltx_align_center ltx_th ltx_th_row"><span class="ltx_text" style="font-size:90%;">2 000 SNPs region</span></th>
<td class="ltx_td ltx_align_center"><math id="S5.T2.m10" class="ltx_Math" alttext="73.91\ [68.95-78.88]" display="inline"><mrow><mn mathsize="90%">73.91</mn><mo lspace="0.450em">⁢</mo><mrow><mo maxsize="90%" minsize="90%">[</mo><mrow><mn mathsize="90%">68.95</mn><mo mathsize="90%">−</mo><mn mathsize="90%">78.88</mn></mrow><mo maxsize="90%" minsize="90%">]</mo></mrow></mrow></math></td>
<td class="ltx_td ltx_align_center"><math id="S5.T2.m11" class="ltx_Math" alttext="100.0\ [99.99-100.0]" display="inline"><mrow><mn mathsize="90%">100.0</mn><mo lspace="0.450em">⁢</mo><mrow><mo maxsize="90%" minsize="90%">[</mo><mrow><mn mathsize="90%">99.99</mn><mo mathsize="90%">−</mo><mn mathsize="90%">100.0</mn></mrow><mo maxsize="90%" minsize="90%">]</mo></mrow></mrow></math></td>
<td class="ltx_td ltx_align_center"><math id="S5.T2.m12" class="ltx_Math" alttext="82.57\ [78.60-86.54]" display="inline"><mrow><mn mathsize="90%">82.57</mn><mo lspace="0.450em">⁢</mo><mrow><mo maxsize="90%" minsize="90%">[</mo><mrow><mn mathsize="90%">78.60</mn><mo mathsize="90%">−</mo><mn mathsize="90%">86.54</mn></mrow><mo maxsize="90%" minsize="90%">]</mo></mrow></mrow></math></td>
<td class="ltx_td ltx_align_center"><math id="S5.T2.m13" class="ltx_Math" alttext="72.03\ [67.02-77.04]" display="inline"><mrow><mn mathsize="90%">72.03</mn><mo lspace="0.450em">⁢</mo><mrow><mo maxsize="90%" minsize="90%">[</mo><mrow><mn mathsize="90%">67.02</mn><mo mathsize="90%">−</mo><mn mathsize="90%">77.04</mn></mrow><mo maxsize="90%" minsize="90%">]</mo></mrow></mrow></math></td>
</tr>
<tr class="ltx_tr">
<th class="ltx_td ltx_align_center ltx_th ltx_th_row"><span class="ltx_text" style="font-size:90%;">800 SNPs regions</span></th>
<td class="ltx_td ltx_align_center"><math id="S5.T2.m14" class="ltx_Math" alttext="80.24\ [75.78-84.69]" display="inline"><mrow><mn mathsize="90%">80.24</mn><mo lspace="0.450em">⁢</mo><mrow><mo maxsize="90%" minsize="90%">[</mo><mrow><mn mathsize="90%">75.78</mn><mo mathsize="90%">−</mo><mn mathsize="90%">84.69</mn></mrow><mo maxsize="90%" minsize="90%">]</mo></mrow></mrow></math></td>
<td class="ltx_td ltx_align_center"><math id="S5.T2.m15" class="ltx_Math" alttext="100.0\ [100.0-100.0]" display="inline"><mrow><mn mathsize="90%">100.0</mn><mo lspace="0.450em">⁢</mo><mrow><mo maxsize="90%" minsize="90%">[</mo><mrow><mn mathsize="90%">100.0</mn><mo mathsize="90%">−</mo><mn mathsize="90%">100.0</mn></mrow><mo maxsize="90%" minsize="90%">]</mo></mrow></mrow></math></td>
<td class="ltx_td ltx_align_center"><math id="S5.T2.m16" class="ltx_Math" alttext="86.07\ [82.54-89.61]" display="inline"><mrow><mn mathsize="90%">86.07</mn><mo lspace="0.450em">⁢</mo><mrow><mo maxsize="90%" minsize="90%">[</mo><mrow><mn mathsize="90%">82.54</mn><mo mathsize="90%">−</mo><mn mathsize="90%">89.61</mn></mrow><mo maxsize="90%" minsize="90%">]</mo></mrow></mrow></math></td>
<td class="ltx_td ltx_align_center"><math id="S5.T2.m17" class="ltx_Math" alttext="80.22\ [75.95-84.48]" display="inline"><mrow><mn mathsize="90%">80.22</mn><mo lspace="0.450em">⁢</mo><mrow><mo maxsize="90%" minsize="90%">[</mo><mrow><mn mathsize="90%">75.95</mn><mo mathsize="90%">−</mo><mn mathsize="90%">84.48</mn></mrow><mo maxsize="90%" minsize="90%">]</mo></mrow></mrow></math></td>
</tr>
<tr class="ltx_tr">
<th class="ltx_td ltx_align_center ltx_th ltx_th_row"><span class="ltx_text" style="font-size:90%;">200 SNPs region</span></th>
<td class="ltx_td ltx_align_center"><math id="S5.T2.m18" class="ltx_Math" alttext="87.62\ [84.26-90.98]" display="inline"><mrow><mn mathsize="90%">87.62</mn><mo lspace="0.450em">⁢</mo><mrow><mo maxsize="90%" minsize="90%">[</mo><mrow><mn mathsize="90%">84.26</mn><mo mathsize="90%">−</mo><mn mathsize="90%">90.98</mn></mrow><mo maxsize="90%" minsize="90%">]</mo></mrow></mrow></math></td>
<td class="ltx_td ltx_align_center"><math id="S5.T2.m19" class="ltx_Math" alttext="100.0\ [100.0-100.0]" display="inline"><mrow><mn mathsize="90%">100.0</mn><mo lspace="0.450em">⁢</mo><mrow><mo maxsize="90%" minsize="90%">[</mo><mrow><mn mathsize="90%">100.0</mn><mo mathsize="90%">−</mo><mn mathsize="90%">100.0</mn></mrow><mo maxsize="90%" minsize="90%">]</mo></mrow></mrow></math></td>
<td class="ltx_td ltx_align_center"><math id="S5.T2.m20" class="ltx_Math" alttext="89.88\ [86.87-92.90]" display="inline"><mrow><mn mathsize="90%">89.88</mn><mo lspace="0.450em">⁢</mo><mrow><mo maxsize="90%" minsize="90%">[</mo><mrow><mn mathsize="90%">86.87</mn><mo mathsize="90%">−</mo><mn mathsize="90%">92.90</mn></mrow><mo maxsize="90%" minsize="90%">]</mo></mrow></mrow></math></td>
<td class="ltx_td ltx_align_center"><math id="S5.T2.m21" class="ltx_Math" alttext="86.65\ [83.16-90.14]" display="inline"><mrow><mn mathsize="90%">86.65</mn><mo lspace="0.450em">⁢</mo><mrow><mo maxsize="90%" minsize="90%">[</mo><mrow><mn mathsize="90%">83.16</mn><mo mathsize="90%">−</mo><mn mathsize="90%">90.14</mn></mrow><mo maxsize="90%" minsize="90%">]</mo></mrow></mrow></math></td>
</tr>
<tr class="ltx_tr">
<th class="ltx_td ltx_align_center ltx_th ltx_th_row ltx_border_b"><span class="ltx_text" style="font-size:90%;">causal SNP</span></th>
<td class="ltx_td ltx_align_center ltx_border_b"><math id="S5.T2.m22" class="ltx_Math" alttext="99.03\ [98.39-99.67]" display="inline"><mrow><mn mathsize="90%">99.03</mn><mo lspace="0.450em">⁢</mo><mrow><mo maxsize="90%" minsize="90%">[</mo><mrow><mn mathsize="90%">98.39</mn><mo mathsize="90%">−</mo><mn mathsize="90%">99.67</mn></mrow><mo maxsize="90%" minsize="90%">]</mo></mrow></mrow></math></td>
<td class="ltx_td ltx_align_center ltx_border_b"><math id="S5.T2.m23" class="ltx_Math" alttext="100.0\ [100.0-100.0]" display="inline"><mrow><mn mathsize="90%">100.0</mn><mo lspace="0.450em">⁢</mo><mrow><mo maxsize="90%" minsize="90%">[</mo><mrow><mn mathsize="90%">100.0</mn><mo mathsize="90%">−</mo><mn mathsize="90%">100.0</mn></mrow><mo maxsize="90%" minsize="90%">]</mo></mrow></mrow></math></td>
<td class="ltx_td ltx_align_center ltx_border_b"><math id="S5.T2.m24" class="ltx_Math" alttext="92.01\ [89.13-94.89]" display="inline"><mrow><mn mathsize="90%">92.01</mn><mo lspace="0.450em">⁢</mo><mrow><mo maxsize="90%" minsize="90%">[</mo><mrow><mn mathsize="90%">89.13</mn><mo mathsize="90%">−</mo><mn mathsize="90%">94.89</mn></mrow><mo maxsize="90%" minsize="90%">]</mo></mrow></mrow></math></td>
<td class="ltx_td ltx_align_center ltx_border_b"><math id="S5.T2.m25" class="ltx_Math" alttext="99.08\ [98.47-99.69]" display="inline"><mrow><mn mathsize="90%">99.08</mn><mo lspace="0.450em">⁢</mo><mrow><mo maxsize="90%" minsize="90%">[</mo><mrow><mn mathsize="90%">98.47</mn><mo mathsize="90%">−</mo><mn mathsize="90%">99.69</mn></mrow><mo maxsize="90%" minsize="90%">]</mo></mrow></mrow></math></td>
</tr>
</tbody>
</table>
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_table">Table 2: </span> Association analysis performed on Chromosome 6 with the four approaches, when the confounding factor <math id="S5.T2.m27" class="ltx_Math" alttext="\tt{treatment}" display="inline"><mi>𝚝𝚛𝚎𝚊𝚝𝚖𝚎𝚗𝚝</mi></math> is considered known. Restricted regions are centered on causal SNP</figcaption>
</figure>
<div class="ltx_pagination ltx_role_newpage"></div>
<figure id="S5.F1" class="ltx_figure"><img src="x1.png" id="S5.F1.g1" class="ltx_graphics ltx_centering ltx_img_square" width="664" height="664" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure">Figure 1: </span>PCA of the genotypes. Scatterplots of the first five principal components. Colors (and symbols) correspond to the 11 human populations in the dataset.</figcaption>
</figure>
<div class="ltx_pagination ltx_role_newpage"></div>
<figure id="S5.F2" class="ltx_figure">
<div class="ltx_flex_figure">
<div class="ltx_flex_cell ltx_flex_size_2">
<figure id="S5.F2.sf1" class="ltx_figure ltx_figure_panel ltx_align_center"><img src="x2.png" id="S5.F2.sf1.g1" class="ltx_graphics ltx_img_square" width="374" height="374" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure"><span class="ltx_text" style="font-size:80%;">(a)</span> </span><span class="ltx_text" style="font-size:80%;">Observing the covariate <math id="S5.F2.sf1.m2" class="ltx_Math" alttext="\tt{treatment}" display="inline"><mi>𝚝𝚛𝚎𝚊𝚝𝚖𝚎𝚗𝚝</mi></math>.</span></figcaption>
</figure>
</div>
<div class="ltx_flex_cell ltx_flex_size_2">
<figure id="S5.F2.sf2" class="ltx_figure ltx_figure_panel ltx_align_center"><img src="x3.png" id="S5.F2.sf2.g1" class="ltx_graphics ltx_img_square" width="374" height="374" alt="Refer to caption">
<figcaption class="ltx_caption"><span class="ltx_tag ltx_tag_figure"><span class="ltx_text" style="font-size:80%;">(b)</span> </span><span class="ltx_text" style="font-size:80%;">Observing the covariate <math id="S5.F2.sf2.m2" class="ltx_Math" alttext="\tt{bmi}" display="inline"><mi>𝚋𝚖𝚒</mi></math>. </span></figcaption>
</figure>
</div>
</div>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 2: </span>Manhattan plots considering Chromosome 6 and observing the covariate <math id="S5.F2.m3" class="ltx_Math" alttext="\tt{treatment}" display="inline"><mi>𝚝𝚛𝚎𝚊𝚝𝚖𝚎𝚗𝚝</mi></math> or the covariate <math id="S5.F2.m4" class="ltx_Math" alttext="\tt{bmi}" display="inline"><mi>𝚋𝚖𝚒</mi></math>. The red vertical line indicates the causal SNP</figcaption>
</figure>
</article>
</div>
<footer class="ltx_page_footer">
<div class="ltx_page_logo">Generated  on Thu Dec  5 02:14:01 2024 by <a href="http://dlmf.nist.gov/LaTeXML/" class="ltx_LaTeXML_logo"><span style="letter-spacing:-0.2em; margin-right:0.1em;">L<span class="ltx_font_smallcaps" style="position:relative; bottom:2.2pt;">a</span>T<span class="ltx_font_smallcaps" style="font-size:120%;position:relative; bottom:-0.2ex;">e</span></span><span style="font-size:90%; position:relative; bottom:-0.2ex;">XML</span><img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAsAAAAOCAYAAAD5YeaVAAAAAXNSR0IArs4c6QAAAAZiS0dEAP8A/wD/oL2nkwAAAAlwSFlzAAALEwAACxMBAJqcGAAAAAd0SU1FB9wKExQZLWTEaOUAAAAddEVYdENvbW1lbnQAQ3JlYXRlZCB3aXRoIFRoZSBHSU1Q72QlbgAAAdpJREFUKM9tkL+L2nAARz9fPZNCKFapUn8kyI0e4iRHSR1Kb8ng0lJw6FYHFwv2LwhOpcWxTjeUunYqOmqd6hEoRDhtDWdA8ApRYsSUCDHNt5ul13vz4w0vWCgUnnEc975arX6ORqN3VqtVZbfbTQC4uEHANM3jSqXymFI6yWazP2KxWAXAL9zCUa1Wy2tXVxheKA9YNoR8Pt+aTqe4FVVVvz05O6MBhqUIBGk8Hn8HAOVy+T+XLJfLS4ZhTiRJgqIoVBRFIoric47jPnmeB1mW/9rr9ZpSSn3Lsmir1fJZlqWlUonKsvwWwD8ymc/nXwVBeLjf7xEKhdBut9Hr9WgmkyGEkJwsy5eHG5vN5g0AKIoCAEgkEkin0wQAfN9/cXPdheu6P33fBwB4ngcAcByHJpPJl+fn54mD3Gg0NrquXxeLRQAAwzAYj8cwTZPwPH9/sVg8PXweDAauqqr2cDjEer1GJBLBZDJBs9mE4zjwfZ85lAGg2+06hmGgXq+j3+/DsixYlgVN03a9Xu8jgCNCyIegIAgx13Vfd7vdu+FweG8YRkjXdWy329+dTgeSJD3ieZ7RNO0VAXAPwDEAO5VKndi2fWrb9jWl9Esul6PZbDY9Go1OZ7PZ9z/lyuD3OozU2wAAAABJRU5ErkJggg==" alt="Mascot Sammy"></a>
</div></footer>
</div>
</body>
</html>
